// telegram-bot-ts-types/build/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var add_sticker_to_set_param_1 = require("./method_params/add_sticker_to_set_param");
exports.AddStickerToSetParam = add_sticker_to_set_param_1.default;
var answer_callback_query_param_1 = require("./method_params/answer_callback_query_param");
exports.AnswerCallbackQueryParam = answer_callback_query_param_1.default;
var answer_inline_query_param_1 = require("./method_params/answer_inline_query_param");
exports.AnswerInlineQueryParam = answer_inline_query_param_1.default;
var answer_pre_checkout_query_param_1 = require("./method_params/answer_pre_checkout_query_param");
exports.AnswerPreCheckoutQueryParam = answer_pre_checkout_query_param_1.default;
var answer_shipping_query_param_1 = require("./method_params/answer_shipping_query_param");
exports.AnswerShippingQueryParam = answer_shipping_query_param_1.default;
var create_new_sticker_set_param_1 = require("./method_params/create_new_sticker_set_param");
exports.CreateNewStickerSetParam = create_new_sticker_set_param_1.default;
var delete_chat_photo_param_1 = require("./method_params/delete_chat_photo_param");
exports.DeleteChatPhotoParam = delete_chat_photo_param_1.default;
var delete_chat_sticker_set_param_1 = require("./method_params/delete_chat_sticker_set_param");
exports.DeleteChatStickerSetParam = delete_chat_sticker_set_param_1.default;
var delete_message_param_1 = require("./method_params/delete_message_param");
exports.DeleteMessageParam = delete_message_param_1.default;
var delete_sticker_from_set_param_1 = require("./method_params/delete_sticker_from_set_param");
exports.DeleteStickerFromSetParam = delete_sticker_from_set_param_1.default;
var delete_webhook_param_1 = require("./method_params/delete_webhook_param");
exports.DeleteWebhookParam = delete_webhook_param_1.default;
var edit_message_caption_param_1 = require("./method_params/edit_message_caption_param");
exports.EditMessageCaptionParam = edit_message_caption_param_1.default;
var edit_message_live_location_param_1 = require("./method_params/edit_message_live_location_param");
exports.EditMessageLiveLocationParam = edit_message_live_location_param_1.default;
var edit_message_media_param_1 = require("./method_params/edit_message_media_param");
exports.EditMessageMediaParam = edit_message_media_param_1.default;
var edit_message_reply_markup_param_1 = require("./method_params/edit_message_reply_markup_param");
exports.EditMessageReplyMarkupParam = edit_message_reply_markup_param_1.default;
var edit_message_text_param_1 = require("./method_params/edit_message_text_param");
exports.EditMessageTextParam = edit_message_text_param_1.default;
var export_chat_invite_link_param_1 = require("./method_params/export_chat_invite_link_param");
exports.ExportChatInviteLinkParam = export_chat_invite_link_param_1.default;
var forward_message_param_1 = require("./method_params/forward_message_param");
exports.ForwardMessageParam = forward_message_param_1.default;
var get_chat_administrators_param_1 = require("./method_params/get_chat_administrators_param");
exports.GetChatAdministratorsParam = get_chat_administrators_param_1.default;
var get_chat_members_count_param_1 = require("./method_params/get_chat_members_count_param");
exports.GetChatMembersCountParam = get_chat_members_count_param_1.default;
var get_chat_member_param_1 = require("./method_params/get_chat_member_param");
exports.GetChatMemberParam = get_chat_member_param_1.default;
var get_chat_param_1 = require("./method_params/get_chat_param");
exports.GetChatParam = get_chat_param_1.default;
var get_file_param_1 = require("./method_params/get_file_param");
exports.GetFileParam = get_file_param_1.default;
var get_game_high_scores_param_1 = require("./method_params/get_game_high_scores_param");
exports.GetGameHighScoresParam = get_game_high_scores_param_1.default;
var get_me_param_1 = require("./method_params/get_me_param");
exports.GetMeParam = get_me_param_1.default;
var get_my_commands_param_1 = require("./method_params/get_my_commands_param");
exports.GetMyCommandsParam = get_my_commands_param_1.default;
var get_sticker_set_param_1 = require("./method_params/get_sticker_set_param");
exports.GetStickerSetParam = get_sticker_set_param_1.default;
var get_updates_param_1 = require("./method_params/get_updates_param");
exports.GetUpdatesParam = get_updates_param_1.default;
var get_user_profile_photos_param_1 = require("./method_params/get_user_profile_photos_param");
exports.GetUserProfilePhotosParam = get_user_profile_photos_param_1.default;
var get_webhook_info_param_1 = require("./method_params/get_webhook_info_param");
exports.GetWebhookInfoParam = get_webhook_info_param_1.default;
var kick_chat_member_param_1 = require("./method_params/kick_chat_member_param");
exports.KickChatMemberParam = kick_chat_member_param_1.default;
var leave_chat_param_1 = require("./method_params/leave_chat_param");
exports.LeaveChatParam = leave_chat_param_1.default;
var pin_chat_message_param_1 = require("./method_params/pin_chat_message_param");
exports.PinChatMessageParam = pin_chat_message_param_1.default;
var promote_chat_member_param_1 = require("./method_params/promote_chat_member_param");
exports.PromoteChatMemberParam = promote_chat_member_param_1.default;
var restrict_chat_member_param_1 = require("./method_params/restrict_chat_member_param");
exports.RestrictChatMemberParam = restrict_chat_member_param_1.default;
var send_animation_param_1 = require("./method_params/send_animation_param");
exports.SendAnimationParam = send_animation_param_1.default;
var send_audio_param_1 = require("./method_params/send_audio_param");
exports.SendAudioParam = send_audio_param_1.default;
var send_chat_action_param_1 = require("./method_params/send_chat_action_param");
exports.SendChatActionParam = send_chat_action_param_1.default;
var send_contact_param_1 = require("./method_params/send_contact_param");
exports.SendContactParam = send_contact_param_1.default;
var send_dice_param_1 = require("./method_params/send_dice_param");
exports.SendDiceParam = send_dice_param_1.default;
var send_document_param_1 = require("./method_params/send_document_param");
exports.SendDocumentParam = send_document_param_1.default;
var send_game_param_1 = require("./method_params/send_game_param");
exports.SendGameParam = send_game_param_1.default;
var send_invoice_param_1 = require("./method_params/send_invoice_param");
exports.SendInvoiceParam = send_invoice_param_1.default;
var send_location_param_1 = require("./method_params/send_location_param");
exports.SendLocationParam = send_location_param_1.default;
var send_media_group_param_1 = require("./method_params/send_media_group_param");
exports.SendMediaGroupParam = send_media_group_param_1.default;
var send_message_param_1 = require("./method_params/send_message_param");
exports.SendMessageParam = send_message_param_1.default;
var send_photo_param_1 = require("./method_params/send_photo_param");
exports.SendPhotoParam = send_photo_param_1.default;
var send_poll_param_1 = require("./method_params/send_poll_param");
exports.SendPollParam = send_poll_param_1.default;
var send_sticker_param_1 = require("./method_params/send_sticker_param");
exports.SendStickerParam = send_sticker_param_1.default;
var send_venue_param_1 = require("./method_params/send_venue_param");
exports.SendVenueParam = send_venue_param_1.default;
var send_video_note_param_1 = require("./method_params/send_video_note_param");
exports.SendVideoNoteParam = send_video_note_param_1.default;
var send_video_param_1 = require("./method_params/send_video_param");
exports.SendVideoParam = send_video_param_1.default;
var send_voice_param_1 = require("./method_params/send_voice_param");
exports.SendVoiceParam = send_voice_param_1.default;
var set_chat_administrator_custom_title_param_1 = require("./method_params/set_chat_administrator_custom_title_param");
exports.SetChatAdministratorCustomTitleParam = set_chat_administrator_custom_title_param_1.default;
var set_chat_description_param_1 = require("./method_params/set_chat_description_param");
exports.SetChatDescriptionParam = set_chat_description_param_1.default;
var set_chat_permissions_param_1 = require("./method_params/set_chat_permissions_param");
exports.SetChatPermissionsParam = set_chat_permissions_param_1.default;
var set_chat_photo_param_1 = require("./method_params/set_chat_photo_param");
exports.SetChatPhotoParam = set_chat_photo_param_1.default;
var set_chat_sticker_set_param_1 = require("./method_params/set_chat_sticker_set_param");
exports.SetChatStickerSetParam = set_chat_sticker_set_param_1.default;
var set_chat_title_param_1 = require("./method_params/set_chat_title_param");
exports.SetChatTitleParam = set_chat_title_param_1.default;
var set_game_score_param_1 = require("./method_params/set_game_score_param");
exports.SetGameScoreParam = set_game_score_param_1.default;
var set_my_commands_param_1 = require("./method_params/set_my_commands_param");
exports.SetMyCommandsParam = set_my_commands_param_1.default;
var set_passport_data_errors_param_1 = require("./method_params/set_passport_data_errors_param");
exports.SetPassportDataErrorsParam = set_passport_data_errors_param_1.default;
var set_sticker_position_in_set_param_1 = require("./method_params/set_sticker_position_in_set_param");
exports.SetStickerPositionInSetParam = set_sticker_position_in_set_param_1.default;
var set_sticker_set_thumb_param_1 = require("./method_params/set_sticker_set_thumb_param");
exports.SetStickerSetThumbParam = set_sticker_set_thumb_param_1.default;
var set_webhook_param_1 = require("./method_params/set_webhook_param");
exports.SetWebhookParam = set_webhook_param_1.default;
var stop_message_live_location_param_1 = require("./method_params/stop_message_live_location_param");
exports.StopMessageLiveLocationParam = stop_message_live_location_param_1.default;
var stop_poll_param_1 = require("./method_params/stop_poll_param");
exports.StopPollParam = stop_poll_param_1.default;
var unban_chat_member_param_1 = require("./method_params/unban_chat_member_param");
exports.UnbanChatMemberParam = unban_chat_member_param_1.default;
var unpin_chat_message_param_1 = require("./method_params/unpin_chat_message_param");
exports.UnpinChatMessageParam = unpin_chat_message_param_1.default;
var upload_sticker_file_param_1 = require("./method_params/upload_sticker_file_param");
exports.UploadStickerFileParam = upload_sticker_file_param_1.default;
var animation_1 = require("./entities/animation");
exports.Animation = animation_1.default;
var audio_1 = require("./entities/audio");
exports.Audio = audio_1.default;
var bot_command_1 = require("./entities/bot_command");
exports.BotCommand = bot_command_1.default;
var callback_game_1 = require("./entities/callback_game");
exports.CallbackGame = callback_game_1.default;
var callback_query_1 = require("./entities/callback_query");
exports.CallbackQuery = callback_query_1.default;
var chat_1 = require("./entities/chat");
exports.Chat = chat_1.default;
var chat_member_1 = require("./entities/chat_member");
exports.ChatMember = chat_member_1.default;
var chat_permissions_1 = require("./entities/chat_permissions");
exports.ChatPermissions = chat_permissions_1.default;
var chat_photo_1 = require("./entities/chat_photo");
exports.ChatPhoto = chat_photo_1.default;
var chosen_inline_result_1 = require("./entities/chosen_inline_result");
exports.ChosenInlineResult = chosen_inline_result_1.default;
var contact_1 = require("./entities/contact");
exports.Contact = contact_1.default;
var dice_1 = require("./entities/dice");
exports.Dice = dice_1.default;
var document_1 = require("./entities/document");
exports.Document = document_1.default;
var encrypted_credentials_1 = require("./entities/encrypted_credentials");
exports.EncryptedCredentials = encrypted_credentials_1.default;
var encrypted_passport_element_1 = require("./entities/encrypted_passport_element");
exports.EncryptedPassportElement = encrypted_passport_element_1.default;
var file_1 = require("./entities/file");
exports.File = file_1.default;
var force_reply_1 = require("./entities/force_reply");
exports.ForceReply = force_reply_1.default;
var game_1 = require("./entities/game");
exports.Game = game_1.default;
var game_high_score_1 = require("./entities/game_high_score");
exports.GameHighScore = game_high_score_1.default;
var inline_keyboard_button_1 = require("./entities/inline_keyboard_button");
exports.InlineKeyboardButton = inline_keyboard_button_1.default;
var inline_keyboard_markup_1 = require("./entities/inline_keyboard_markup");
exports.InlineKeyboardMarkup = inline_keyboard_markup_1.default;
var inline_query_1 = require("./entities/inline_query");
exports.InlineQuery = inline_query_1.default;
var inline_query_result_1 = require("./entities/inline_query_result");
exports.InlineQueryResult = inline_query_result_1.default;
var inline_query_result_article_1 = require("./entities/inline_query_result_article");
exports.InlineQueryResultArticle = inline_query_result_article_1.default;
var inline_query_result_audio_1 = require("./entities/inline_query_result_audio");
exports.InlineQueryResultAudio = inline_query_result_audio_1.default;
var inline_query_result_cached_audio_1 = require("./entities/inline_query_result_cached_audio");
exports.InlineQueryResultCachedAudio = inline_query_result_cached_audio_1.default;
var inline_query_result_cached_document_1 = require("./entities/inline_query_result_cached_document");
exports.InlineQueryResultCachedDocument = inline_query_result_cached_document_1.default;
var inline_query_result_cached_gif_1 = require("./entities/inline_query_result_cached_gif");
exports.InlineQueryResultCachedGif = inline_query_result_cached_gif_1.default;
var inline_query_result_cached_mpeg4_gif_1 = require("./entities/inline_query_result_cached_mpeg4_gif");
exports.InlineQueryResultCachedMpeg4Gif = inline_query_result_cached_mpeg4_gif_1.default;
var inline_query_result_cached_photo_1 = require("./entities/inline_query_result_cached_photo");
exports.InlineQueryResultCachedPhoto = inline_query_result_cached_photo_1.default;
var inline_query_result_cached_sticker_1 = require("./entities/inline_query_result_cached_sticker");
exports.InlineQueryResultCachedSticker = inline_query_result_cached_sticker_1.default;
var inline_query_result_cached_video_1 = require("./entities/inline_query_result_cached_video");
exports.InlineQueryResultCachedVideo = inline_query_result_cached_video_1.default;
var inline_query_result_cached_voice_1 = require("./entities/inline_query_result_cached_voice");
exports.InlineQueryResultCachedVoice = inline_query_result_cached_voice_1.default;
var inline_query_result_contact_1 = require("./entities/inline_query_result_contact");
exports.InlineQueryResultContact = inline_query_result_contact_1.default;
var inline_query_result_document_1 = require("./entities/inline_query_result_document");
exports.InlineQueryResultDocument = inline_query_result_document_1.default;
var inline_query_result_game_1 = require("./entities/inline_query_result_game");
exports.InlineQueryResultGame = inline_query_result_game_1.default;
var inline_query_result_gif_1 = require("./entities/inline_query_result_gif");
exports.InlineQueryResultGif = inline_query_result_gif_1.default;
var inline_query_result_location_1 = require("./entities/inline_query_result_location");
exports.InlineQueryResultLocation = inline_query_result_location_1.default;
var inline_query_result_mpeg4_gif_1 = require("./entities/inline_query_result_mpeg4_gif");
exports.InlineQueryResultMpeg4Gif = inline_query_result_mpeg4_gif_1.default;
var inline_query_result_photo_1 = require("./entities/inline_query_result_photo");
exports.InlineQueryResultPhoto = inline_query_result_photo_1.default;
var inline_query_result_venue_1 = require("./entities/inline_query_result_venue");
exports.InlineQueryResultVenue = inline_query_result_venue_1.default;
var inline_query_result_video_1 = require("./entities/inline_query_result_video");
exports.InlineQueryResultVideo = inline_query_result_video_1.default;
var inline_query_result_voice_1 = require("./entities/inline_query_result_voice");
exports.InlineQueryResultVoice = inline_query_result_voice_1.default;
var input_contact_message_content_1 = require("./entities/input_contact_message_content");
exports.InputContactMessageContent = input_contact_message_content_1.default;
var input_file_1 = require("./entities/input_file");
exports.InputFile = input_file_1.default;
var input_location_message_content_1 = require("./entities/input_location_message_content");
exports.InputLocationMessageContent = input_location_message_content_1.default;
var input_media_1 = require("./entities/input_media");
exports.InputMedia = input_media_1.default;
var input_media_animation_1 = require("./entities/input_media_animation");
exports.InputMediaAnimation = input_media_animation_1.default;
var input_media_audio_1 = require("./entities/input_media_audio");
exports.InputMediaAudio = input_media_audio_1.default;
var input_media_document_1 = require("./entities/input_media_document");
exports.InputMediaDocument = input_media_document_1.default;
var input_media_photo_1 = require("./entities/input_media_photo");
exports.InputMediaPhoto = input_media_photo_1.default;
var input_media_video_1 = require("./entities/input_media_video");
exports.InputMediaVideo = input_media_video_1.default;
var input_message_content_1 = require("./entities/input_message_content");
exports.InputMessageContent = input_message_content_1.default;
var input_text_message_content_1 = require("./entities/input_text_message_content");
exports.InputTextMessageContent = input_text_message_content_1.default;
var input_venue_message_content_1 = require("./entities/input_venue_message_content");
exports.InputVenueMessageContent = input_venue_message_content_1.default;
var invoice_1 = require("./entities/invoice");
exports.Invoice = invoice_1.default;
var keyboard_button_1 = require("./entities/keyboard_button");
exports.KeyboardButton = keyboard_button_1.default;
var keyboard_button_poll_type_1 = require("./entities/keyboard_button_poll_type");
exports.KeyboardButtonPollType = keyboard_button_poll_type_1.default;
var labeled_price_1 = require("./entities/labeled_price");
exports.LabeledPrice = labeled_price_1.default;
var location_1 = require("./entities/location");
exports.Location = location_1.default;
var login_url_1 = require("./entities/login_url");
exports.LoginUrl = login_url_1.default;
var mask_position_1 = require("./entities/mask_position");
exports.MaskPosition = mask_position_1.default;
var message_1 = require("./entities/message");
exports.Message = message_1.default;
var message_entity_1 = require("./entities/message_entity");
exports.MessageEntity = message_entity_1.default;
var order_info_1 = require("./entities/order_info");
exports.OrderInfo = order_info_1.default;
var passport_data_1 = require("./entities/passport_data");
exports.PassportData = passport_data_1.default;
var passport_element_error_1 = require("./entities/passport_element_error");
exports.PassportElementError = passport_element_error_1.default;
var passport_element_error_data_field_1 = require("./entities/passport_element_error_data_field");
exports.PassportElementErrorDataField = passport_element_error_data_field_1.default;
var passport_element_error_file_1 = require("./entities/passport_element_error_file");
exports.PassportElementErrorFile = passport_element_error_file_1.default;
var passport_element_error_files_1 = require("./entities/passport_element_error_files");
exports.PassportElementErrorFiles = passport_element_error_files_1.default;
var passport_element_error_front_side_1 = require("./entities/passport_element_error_front_side");
exports.PassportElementErrorFrontSide = passport_element_error_front_side_1.default;
var passport_element_error_reverse_side_1 = require("./entities/passport_element_error_reverse_side");
exports.PassportElementErrorReverseSide = passport_element_error_reverse_side_1.default;
var passport_element_error_selfie_1 = require("./entities/passport_element_error_selfie");
exports.PassportElementErrorSelfie = passport_element_error_selfie_1.default;
var passport_element_error_translation_file_1 = require("./entities/passport_element_error_translation_file");
exports.PassportElementErrorTranslationFile = passport_element_error_translation_file_1.default;
var passport_element_error_translation_files_1 = require("./entities/passport_element_error_translation_files");
exports.PassportElementErrorTranslationFiles = passport_element_error_translation_files_1.default;
var passport_element_error_unspecified_1 = require("./entities/passport_element_error_unspecified");
exports.PassportElementErrorUnspecified = passport_element_error_unspecified_1.default;
var passport_file_1 = require("./entities/passport_file");
exports.PassportFile = passport_file_1.default;
var photo_size_1 = require("./entities/photo_size");
exports.PhotoSize = photo_size_1.default;
var poll_1 = require("./entities/poll");
exports.Poll = poll_1.default;
var poll_answer_1 = require("./entities/poll_answer");
exports.PollAnswer = poll_answer_1.default;
var poll_option_1 = require("./entities/poll_option");
exports.PollOption = poll_option_1.default;
var pre_checkout_query_1 = require("./entities/pre_checkout_query");
exports.PreCheckoutQuery = pre_checkout_query_1.default;
var reply_keyboard_markup_1 = require("./entities/reply_keyboard_markup");
exports.ReplyKeyboardMarkup = reply_keyboard_markup_1.default;
var reply_keyboard_remove_1 = require("./entities/reply_keyboard_remove");
exports.ReplyKeyboardRemove = reply_keyboard_remove_1.default;
var response_parameters_1 = require("./entities/response_parameters");
exports.ResponseParameters = response_parameters_1.default;
var shipping_address_1 = require("./entities/shipping_address");
exports.ShippingAddress = shipping_address_1.default;
var shipping_option_1 = require("./entities/shipping_option");
exports.ShippingOption = shipping_option_1.default;
var shipping_query_1 = require("./entities/shipping_query");
exports.ShippingQuery = shipping_query_1.default;
var sticker_1 = require("./entities/sticker");
exports.Sticker = sticker_1.default;
var sticker_set_1 = require("./entities/sticker_set");
exports.StickerSet = sticker_set_1.default;
var successful_payment_1 = require("./entities/successful_payment");
exports.SuccessfulPayment = successful_payment_1.default;
var update_1 = require("./entities/update");
exports.Update = update_1.default;
var user_1 = require("./entities/user");
exports.User = user_1.default;
var user_profile_photos_1 = require("./entities/user_profile_photos");
exports.UserProfilePhotos = user_profile_photos_1.default;
var venue_1 = require("./entities/venue");
exports.Venue = venue_1.default;
var video_1 = require("./entities/video");
exports.Video = video_1.default;
var video_note_1 = require("./entities/video_note");
exports.VideoNote = video_note_1.default;
var voice_1 = require("./entities/voice");
exports.Voice = voice_1.default;
var webhook_info_1 = require("./entities/webhook_info");
exports.WebhookInfo = webhook_info_1.default;
var add_sticker_to_set_param_serializer_1 = require("./serialize/add_sticker_to_set_param_serializer");
exports.AddStickerToSetParamSerializer = add_sticker_to_set_param_serializer_1.AddStickerToSetParamSerializer;
exports.AddStickerToSetParamSerializerParams = add_sticker_to_set_param_serializer_1.AddStickerToSetParamSerializerParams;
var animation_serializer_1 = require("./serialize/animation_serializer");
exports.AnimationSerializer = animation_serializer_1.AnimationSerializer;
exports.AnimationSerializerParams = animation_serializer_1.AnimationSerializerParams;
var answer_callback_query_param_serializer_1 = require("./serialize/answer_callback_query_param_serializer");
exports.AnswerCallbackQueryParamSerializer = answer_callback_query_param_serializer_1.AnswerCallbackQueryParamSerializer;
exports.AnswerCallbackQueryParamSerializerParams = answer_callback_query_param_serializer_1.AnswerCallbackQueryParamSerializerParams;
var answer_inline_query_param_serializer_1 = require("./serialize/answer_inline_query_param_serializer");
exports.AnswerInlineQueryParamSerializer = answer_inline_query_param_serializer_1.AnswerInlineQueryParamSerializer;
exports.AnswerInlineQueryParamSerializerParams = answer_inline_query_param_serializer_1.AnswerInlineQueryParamSerializerParams;
var answer_pre_checkout_query_param_serializer_1 = require("./serialize/answer_pre_checkout_query_param_serializer");
exports.AnswerPreCheckoutQueryParamSerializer = answer_pre_checkout_query_param_serializer_1.AnswerPreCheckoutQueryParamSerializer;
exports.AnswerPreCheckoutQueryParamSerializerParams = answer_pre_checkout_query_param_serializer_1.AnswerPreCheckoutQueryParamSerializerParams;
var answer_shipping_query_param_serializer_1 = require("./serialize/answer_shipping_query_param_serializer");
exports.AnswerShippingQueryParamSerializer = answer_shipping_query_param_serializer_1.AnswerShippingQueryParamSerializer;
exports.AnswerShippingQueryParamSerializerParams = answer_shipping_query_param_serializer_1.AnswerShippingQueryParamSerializerParams;
var audio_serializer_1 = require("./serialize/audio_serializer");
exports.AudioSerializer = audio_serializer_1.AudioSerializer;
exports.AudioSerializerParams = audio_serializer_1.AudioSerializerParams;
var bot_command_serializer_1 = require("./serialize/bot_command_serializer");
exports.BotCommandSerializer = bot_command_serializer_1.BotCommandSerializer;
exports.BotCommandSerializerParams = bot_command_serializer_1.BotCommandSerializerParams;
var callback_game_serializer_1 = require("./serialize/callback_game_serializer");
exports.CallbackGameSerializer = callback_game_serializer_1.CallbackGameSerializer;
exports.CallbackGameSerializerParams = callback_game_serializer_1.CallbackGameSerializerParams;
var callback_query_serializer_1 = require("./serialize/callback_query_serializer");
exports.CallbackQuerySerializer = callback_query_serializer_1.CallbackQuerySerializer;
exports.CallbackQuerySerializerParams = callback_query_serializer_1.CallbackQuerySerializerParams;
var chat_member_serializer_1 = require("./serialize/chat_member_serializer");
exports.ChatMemberSerializer = chat_member_serializer_1.ChatMemberSerializer;
exports.ChatMemberSerializerParams = chat_member_serializer_1.ChatMemberSerializerParams;
var chat_permissions_serializer_1 = require("./serialize/chat_permissions_serializer");
exports.ChatPermissionsSerializer = chat_permissions_serializer_1.ChatPermissionsSerializer;
exports.ChatPermissionsSerializerParams = chat_permissions_serializer_1.ChatPermissionsSerializerParams;
var chat_photo_serializer_1 = require("./serialize/chat_photo_serializer");
exports.ChatPhotoSerializer = chat_photo_serializer_1.ChatPhotoSerializer;
exports.ChatPhotoSerializerParams = chat_photo_serializer_1.ChatPhotoSerializerParams;
var chat_serializer_1 = require("./serialize/chat_serializer");
exports.ChatSerializer = chat_serializer_1.ChatSerializer;
exports.ChatSerializerParams = chat_serializer_1.ChatSerializerParams;
var chosen_inline_result_serializer_1 = require("./serialize/chosen_inline_result_serializer");
exports.ChosenInlineResultSerializer = chosen_inline_result_serializer_1.ChosenInlineResultSerializer;
exports.ChosenInlineResultSerializerParams = chosen_inline_result_serializer_1.ChosenInlineResultSerializerParams;
var contact_serializer_1 = require("./serialize/contact_serializer");
exports.ContactSerializer = contact_serializer_1.ContactSerializer;
exports.ContactSerializerParams = contact_serializer_1.ContactSerializerParams;
var create_new_sticker_set_param_serializer_1 = require("./serialize/create_new_sticker_set_param_serializer");
exports.CreateNewStickerSetParamSerializer = create_new_sticker_set_param_serializer_1.CreateNewStickerSetParamSerializer;
exports.CreateNewStickerSetParamSerializerParams = create_new_sticker_set_param_serializer_1.CreateNewStickerSetParamSerializerParams;
var delete_chat_photo_param_serializer_1 = require("./serialize/delete_chat_photo_param_serializer");
exports.DeleteChatPhotoParamSerializer = delete_chat_photo_param_serializer_1.DeleteChatPhotoParamSerializer;
exports.DeleteChatPhotoParamSerializerParams = delete_chat_photo_param_serializer_1.DeleteChatPhotoParamSerializerParams;
var delete_chat_sticker_set_param_serializer_1 = require("./serialize/delete_chat_sticker_set_param_serializer");
exports.DeleteChatStickerSetParamSerializer = delete_chat_sticker_set_param_serializer_1.DeleteChatStickerSetParamSerializer;
exports.DeleteChatStickerSetParamSerializerParams = delete_chat_sticker_set_param_serializer_1.DeleteChatStickerSetParamSerializerParams;
var delete_message_param_serializer_1 = require("./serialize/delete_message_param_serializer");
exports.DeleteMessageParamSerializer = delete_message_param_serializer_1.DeleteMessageParamSerializer;
exports.DeleteMessageParamSerializerParams = delete_message_param_serializer_1.DeleteMessageParamSerializerParams;
var delete_sticker_from_set_param_serializer_1 = require("./serialize/delete_sticker_from_set_param_serializer");
exports.DeleteStickerFromSetParamSerializer = delete_sticker_from_set_param_serializer_1.DeleteStickerFromSetParamSerializer;
exports.DeleteStickerFromSetParamSerializerParams = delete_sticker_from_set_param_serializer_1.DeleteStickerFromSetParamSerializerParams;
var delete_webhook_param_serializer_1 = require("./serialize/delete_webhook_param_serializer");
exports.DeleteWebhookParamSerializer = delete_webhook_param_serializer_1.DeleteWebhookParamSerializer;
exports.DeleteWebhookParamSerializerParams = delete_webhook_param_serializer_1.DeleteWebhookParamSerializerParams;
var dice_serializer_1 = require("./serialize/dice_serializer");
exports.DiceSerializer = dice_serializer_1.DiceSerializer;
exports.DiceSerializerParams = dice_serializer_1.DiceSerializerParams;
var document_serializer_1 = require("./serialize/document_serializer");
exports.DocumentSerializer = document_serializer_1.DocumentSerializer;
exports.DocumentSerializerParams = document_serializer_1.DocumentSerializerParams;
var edit_message_caption_param_serializer_1 = require("./serialize/edit_message_caption_param_serializer");
exports.EditMessageCaptionParamSerializer = edit_message_caption_param_serializer_1.EditMessageCaptionParamSerializer;
exports.EditMessageCaptionParamSerializerParams = edit_message_caption_param_serializer_1.EditMessageCaptionParamSerializerParams;
var edit_message_live_location_param_serializer_1 = require("./serialize/edit_message_live_location_param_serializer");
exports.EditMessageLiveLocationParamSerializer = edit_message_live_location_param_serializer_1.EditMessageLiveLocationParamSerializer;
exports.EditMessageLiveLocationParamSerializerParams = edit_message_live_location_param_serializer_1.EditMessageLiveLocationParamSerializerParams;
var edit_message_media_param_serializer_1 = require("./serialize/edit_message_media_param_serializer");
exports.EditMessageMediaParamSerializer = edit_message_media_param_serializer_1.EditMessageMediaParamSerializer;
exports.EditMessageMediaParamSerializerParams = edit_message_media_param_serializer_1.EditMessageMediaParamSerializerParams;
var edit_message_reply_markup_param_serializer_1 = require("./serialize/edit_message_reply_markup_param_serializer");
exports.EditMessageReplyMarkupParamSerializer = edit_message_reply_markup_param_serializer_1.EditMessageReplyMarkupParamSerializer;
exports.EditMessageReplyMarkupParamSerializerParams = edit_message_reply_markup_param_serializer_1.EditMessageReplyMarkupParamSerializerParams;
var edit_message_text_param_serializer_1 = require("./serialize/edit_message_text_param_serializer");
exports.EditMessageTextParamSerializer = edit_message_text_param_serializer_1.EditMessageTextParamSerializer;
exports.EditMessageTextParamSerializerParams = edit_message_text_param_serializer_1.EditMessageTextParamSerializerParams;
var encrypted_credentials_serializer_1 = require("./serialize/encrypted_credentials_serializer");
exports.EncryptedCredentialsSerializer = encrypted_credentials_serializer_1.EncryptedCredentialsSerializer;
exports.EncryptedCredentialsSerializerParams = encrypted_credentials_serializer_1.EncryptedCredentialsSerializerParams;
var encrypted_passport_element_serializer_1 = require("./serialize/encrypted_passport_element_serializer");
exports.EncryptedPassportElementSerializer = encrypted_passport_element_serializer_1.EncryptedPassportElementSerializer;
exports.EncryptedPassportElementSerializerParams = encrypted_passport_element_serializer_1.EncryptedPassportElementSerializerParams;
var export_chat_invite_link_param_serializer_1 = require("./serialize/export_chat_invite_link_param_serializer");
exports.ExportChatInviteLinkParamSerializer = export_chat_invite_link_param_serializer_1.ExportChatInviteLinkParamSerializer;
exports.ExportChatInviteLinkParamSerializerParams = export_chat_invite_link_param_serializer_1.ExportChatInviteLinkParamSerializerParams;
var file_serializer_1 = require("./serialize/file_serializer");
exports.FileSerializer = file_serializer_1.FileSerializer;
exports.FileSerializerParams = file_serializer_1.FileSerializerParams;
var force_reply_serializer_1 = require("./serialize/force_reply_serializer");
exports.ForceReplySerializer = force_reply_serializer_1.ForceReplySerializer;
exports.ForceReplySerializerParams = force_reply_serializer_1.ForceReplySerializerParams;
var forward_message_param_serializer_1 = require("./serialize/forward_message_param_serializer");
exports.ForwardMessageParamSerializer = forward_message_param_serializer_1.ForwardMessageParamSerializer;
exports.ForwardMessageParamSerializerParams = forward_message_param_serializer_1.ForwardMessageParamSerializerParams;
var game_high_score_serializer_1 = require("./serialize/game_high_score_serializer");
exports.GameHighScoreSerializer = game_high_score_serializer_1.GameHighScoreSerializer;
exports.GameHighScoreSerializerParams = game_high_score_serializer_1.GameHighScoreSerializerParams;
var game_serializer_1 = require("./serialize/game_serializer");
exports.GameSerializer = game_serializer_1.GameSerializer;
exports.GameSerializerParams = game_serializer_1.GameSerializerParams;
var get_chat_administrators_param_serializer_1 = require("./serialize/get_chat_administrators_param_serializer");
exports.GetChatAdministratorsParamSerializer = get_chat_administrators_param_serializer_1.GetChatAdministratorsParamSerializer;
exports.GetChatAdministratorsParamSerializerParams = get_chat_administrators_param_serializer_1.GetChatAdministratorsParamSerializerParams;
var get_chat_members_count_param_serializer_1 = require("./serialize/get_chat_members_count_param_serializer");
exports.GetChatMembersCountParamSerializer = get_chat_members_count_param_serializer_1.GetChatMembersCountParamSerializer;
exports.GetChatMembersCountParamSerializerParams = get_chat_members_count_param_serializer_1.GetChatMembersCountParamSerializerParams;
var get_chat_member_param_serializer_1 = require("./serialize/get_chat_member_param_serializer");
exports.GetChatMemberParamSerializer = get_chat_member_param_serializer_1.GetChatMemberParamSerializer;
exports.GetChatMemberParamSerializerParams = get_chat_member_param_serializer_1.GetChatMemberParamSerializerParams;
var get_chat_param_serializer_1 = require("./serialize/get_chat_param_serializer");
exports.GetChatParamSerializer = get_chat_param_serializer_1.GetChatParamSerializer;
exports.GetChatParamSerializerParams = get_chat_param_serializer_1.GetChatParamSerializerParams;
var get_file_param_serializer_1 = require("./serialize/get_file_param_serializer");
exports.GetFileParamSerializer = get_file_param_serializer_1.GetFileParamSerializer;
exports.GetFileParamSerializerParams = get_file_param_serializer_1.GetFileParamSerializerParams;
var get_game_high_scores_param_serializer_1 = require("./serialize/get_game_high_scores_param_serializer");
exports.GetGameHighScoresParamSerializer = get_game_high_scores_param_serializer_1.GetGameHighScoresParamSerializer;
exports.GetGameHighScoresParamSerializerParams = get_game_high_scores_param_serializer_1.GetGameHighScoresParamSerializerParams;
var get_me_param_serializer_1 = require("./serialize/get_me_param_serializer");
exports.GetMeParamSerializer = get_me_param_serializer_1.GetMeParamSerializer;
exports.GetMeParamSerializerParams = get_me_param_serializer_1.GetMeParamSerializerParams;
var get_my_commands_param_serializer_1 = require("./serialize/get_my_commands_param_serializer");
exports.GetMyCommandsParamSerializer = get_my_commands_param_serializer_1.GetMyCommandsParamSerializer;
exports.GetMyCommandsParamSerializerParams = get_my_commands_param_serializer_1.GetMyCommandsParamSerializerParams;
var get_sticker_set_param_serializer_1 = require("./serialize/get_sticker_set_param_serializer");
exports.GetStickerSetParamSerializer = get_sticker_set_param_serializer_1.GetStickerSetParamSerializer;
exports.GetStickerSetParamSerializerParams = get_sticker_set_param_serializer_1.GetStickerSetParamSerializerParams;
var get_updates_param_serializer_1 = require("./serialize/get_updates_param_serializer");
exports.GetUpdatesParamSerializer = get_updates_param_serializer_1.GetUpdatesParamSerializer;
exports.GetUpdatesParamSerializerParams = get_updates_param_serializer_1.GetUpdatesParamSerializerParams;
var get_user_profile_photos_param_serializer_1 = require("./serialize/get_user_profile_photos_param_serializer");
exports.GetUserProfilePhotosParamSerializer = get_user_profile_photos_param_serializer_1.GetUserProfilePhotosParamSerializer;
exports.GetUserProfilePhotosParamSerializerParams = get_user_profile_photos_param_serializer_1.GetUserProfilePhotosParamSerializerParams;
var get_webhook_info_param_serializer_1 = require("./serialize/get_webhook_info_param_serializer");
exports.GetWebhookInfoParamSerializer = get_webhook_info_param_serializer_1.GetWebhookInfoParamSerializer;
exports.GetWebhookInfoParamSerializerParams = get_webhook_info_param_serializer_1.GetWebhookInfoParamSerializerParams;
var inline_keyboard_button_serializer_1 = require("./serialize/inline_keyboard_button_serializer");
exports.InlineKeyboardButtonSerializer = inline_keyboard_button_serializer_1.InlineKeyboardButtonSerializer;
exports.InlineKeyboardButtonSerializerParams = inline_keyboard_button_serializer_1.InlineKeyboardButtonSerializerParams;
var inline_keyboard_markup_serializer_1 = require("./serialize/inline_keyboard_markup_serializer");
exports.InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.InlineKeyboardMarkupSerializer;
exports.InlineKeyboardMarkupSerializerParams = inline_keyboard_markup_serializer_1.InlineKeyboardMarkupSerializerParams;
var inline_query_result_article_serializer_1 = require("./serialize/inline_query_result_article_serializer");
exports.InlineQueryResultArticleSerializer = inline_query_result_article_serializer_1.InlineQueryResultArticleSerializer;
exports.InlineQueryResultArticleSerializerParams = inline_query_result_article_serializer_1.InlineQueryResultArticleSerializerParams;
var inline_query_result_audio_serializer_1 = require("./serialize/inline_query_result_audio_serializer");
exports.InlineQueryResultAudioSerializer = inline_query_result_audio_serializer_1.InlineQueryResultAudioSerializer;
exports.InlineQueryResultAudioSerializerParams = inline_query_result_audio_serializer_1.InlineQueryResultAudioSerializerParams;
var inline_query_result_cached_audio_serializer_1 = require("./serialize/inline_query_result_cached_audio_serializer");
exports.InlineQueryResultCachedAudioSerializer = inline_query_result_cached_audio_serializer_1.InlineQueryResultCachedAudioSerializer;
exports.InlineQueryResultCachedAudioSerializerParams = inline_query_result_cached_audio_serializer_1.InlineQueryResultCachedAudioSerializerParams;
var inline_query_result_cached_document_serializer_1 = require("./serialize/inline_query_result_cached_document_serializer");
exports.InlineQueryResultCachedDocumentSerializer = inline_query_result_cached_document_serializer_1.InlineQueryResultCachedDocumentSerializer;
exports.InlineQueryResultCachedDocumentSerializerParams = inline_query_result_cached_document_serializer_1.InlineQueryResultCachedDocumentSerializerParams;
var inline_query_result_cached_gif_serializer_1 = require("./serialize/inline_query_result_cached_gif_serializer");
exports.InlineQueryResultCachedGifSerializer = inline_query_result_cached_gif_serializer_1.InlineQueryResultCachedGifSerializer;
exports.InlineQueryResultCachedGifSerializerParams = inline_query_result_cached_gif_serializer_1.InlineQueryResultCachedGifSerializerParams;
var inline_query_result_cached_mpeg4_gif_serializer_1 = require("./serialize/inline_query_result_cached_mpeg4_gif_serializer");
exports.InlineQueryResultCachedMpeg4GifSerializer = inline_query_result_cached_mpeg4_gif_serializer_1.InlineQueryResultCachedMpeg4GifSerializer;
exports.InlineQueryResultCachedMpeg4GifSerializerParams = inline_query_result_cached_mpeg4_gif_serializer_1.InlineQueryResultCachedMpeg4GifSerializerParams;
var inline_query_result_cached_photo_serializer_1 = require("./serialize/inline_query_result_cached_photo_serializer");
exports.InlineQueryResultCachedPhotoSerializer = inline_query_result_cached_photo_serializer_1.InlineQueryResultCachedPhotoSerializer;
exports.InlineQueryResultCachedPhotoSerializerParams = inline_query_result_cached_photo_serializer_1.InlineQueryResultCachedPhotoSerializerParams;
var inline_query_result_cached_sticker_serializer_1 = require("./serialize/inline_query_result_cached_sticker_serializer");
exports.InlineQueryResultCachedStickerSerializer = inline_query_result_cached_sticker_serializer_1.InlineQueryResultCachedStickerSerializer;
exports.InlineQueryResultCachedStickerSerializerParams = inline_query_result_cached_sticker_serializer_1.InlineQueryResultCachedStickerSerializerParams;
var inline_query_result_cached_video_serializer_1 = require("./serialize/inline_query_result_cached_video_serializer");
exports.InlineQueryResultCachedVideoSerializer = inline_query_result_cached_video_serializer_1.InlineQueryResultCachedVideoSerializer;
exports.InlineQueryResultCachedVideoSerializerParams = inline_query_result_cached_video_serializer_1.InlineQueryResultCachedVideoSerializerParams;
var inline_query_result_cached_voice_serializer_1 = require("./serialize/inline_query_result_cached_voice_serializer");
exports.InlineQueryResultCachedVoiceSerializer = inline_query_result_cached_voice_serializer_1.InlineQueryResultCachedVoiceSerializer;
exports.InlineQueryResultCachedVoiceSerializerParams = inline_query_result_cached_voice_serializer_1.InlineQueryResultCachedVoiceSerializerParams;
var inline_query_result_contact_serializer_1 = require("./serialize/inline_query_result_contact_serializer");
exports.InlineQueryResultContactSerializer = inline_query_result_contact_serializer_1.InlineQueryResultContactSerializer;
exports.InlineQueryResultContactSerializerParams = inline_query_result_contact_serializer_1.InlineQueryResultContactSerializerParams;
var inline_query_result_document_serializer_1 = require("./serialize/inline_query_result_document_serializer");
exports.InlineQueryResultDocumentSerializer = inline_query_result_document_serializer_1.InlineQueryResultDocumentSerializer;
exports.InlineQueryResultDocumentSerializerParams = inline_query_result_document_serializer_1.InlineQueryResultDocumentSerializerParams;
var inline_query_result_game_serializer_1 = require("./serialize/inline_query_result_game_serializer");
exports.InlineQueryResultGameSerializer = inline_query_result_game_serializer_1.InlineQueryResultGameSerializer;
exports.InlineQueryResultGameSerializerParams = inline_query_result_game_serializer_1.InlineQueryResultGameSerializerParams;
var inline_query_result_gif_serializer_1 = require("./serialize/inline_query_result_gif_serializer");
exports.InlineQueryResultGifSerializer = inline_query_result_gif_serializer_1.InlineQueryResultGifSerializer;
exports.InlineQueryResultGifSerializerParams = inline_query_result_gif_serializer_1.InlineQueryResultGifSerializerParams;
var inline_query_result_location_serializer_1 = require("./serialize/inline_query_result_location_serializer");
exports.InlineQueryResultLocationSerializer = inline_query_result_location_serializer_1.InlineQueryResultLocationSerializer;
exports.InlineQueryResultLocationSerializerParams = inline_query_result_location_serializer_1.InlineQueryResultLocationSerializerParams;
var inline_query_result_mpeg4_gif_serializer_1 = require("./serialize/inline_query_result_mpeg4_gif_serializer");
exports.InlineQueryResultMpeg4GifSerializer = inline_query_result_mpeg4_gif_serializer_1.InlineQueryResultMpeg4GifSerializer;
exports.InlineQueryResultMpeg4GifSerializerParams = inline_query_result_mpeg4_gif_serializer_1.InlineQueryResultMpeg4GifSerializerParams;
var inline_query_result_photo_serializer_1 = require("./serialize/inline_query_result_photo_serializer");
exports.InlineQueryResultPhotoSerializer = inline_query_result_photo_serializer_1.InlineQueryResultPhotoSerializer;
exports.InlineQueryResultPhotoSerializerParams = inline_query_result_photo_serializer_1.InlineQueryResultPhotoSerializerParams;
var inline_query_result_serializer_1 = require("./serialize/inline_query_result_serializer");
exports.InlineQueryResultSerializer = inline_query_result_serializer_1.InlineQueryResultSerializer;
exports.InlineQueryResultSerializerParams = inline_query_result_serializer_1.InlineQueryResultSerializerParams;
var inline_query_result_venue_serializer_1 = require("./serialize/inline_query_result_venue_serializer");
exports.InlineQueryResultVenueSerializer = inline_query_result_venue_serializer_1.InlineQueryResultVenueSerializer;
exports.InlineQueryResultVenueSerializerParams = inline_query_result_venue_serializer_1.InlineQueryResultVenueSerializerParams;
var inline_query_result_video_serializer_1 = require("./serialize/inline_query_result_video_serializer");
exports.InlineQueryResultVideoSerializer = inline_query_result_video_serializer_1.InlineQueryResultVideoSerializer;
exports.InlineQueryResultVideoSerializerParams = inline_query_result_video_serializer_1.InlineQueryResultVideoSerializerParams;
var inline_query_result_voice_serializer_1 = require("./serialize/inline_query_result_voice_serializer");
exports.InlineQueryResultVoiceSerializer = inline_query_result_voice_serializer_1.InlineQueryResultVoiceSerializer;
exports.InlineQueryResultVoiceSerializerParams = inline_query_result_voice_serializer_1.InlineQueryResultVoiceSerializerParams;
var inline_query_serializer_1 = require("./serialize/inline_query_serializer");
exports.InlineQuerySerializer = inline_query_serializer_1.InlineQuerySerializer;
exports.InlineQuerySerializerParams = inline_query_serializer_1.InlineQuerySerializerParams;
var input_contact_message_content_serializer_1 = require("./serialize/input_contact_message_content_serializer");
exports.InputContactMessageContentSerializer = input_contact_message_content_serializer_1.InputContactMessageContentSerializer;
exports.InputContactMessageContentSerializerParams = input_contact_message_content_serializer_1.InputContactMessageContentSerializerParams;
var input_location_message_content_serializer_1 = require("./serialize/input_location_message_content_serializer");
exports.InputLocationMessageContentSerializer = input_location_message_content_serializer_1.InputLocationMessageContentSerializer;
exports.InputLocationMessageContentSerializerParams = input_location_message_content_serializer_1.InputLocationMessageContentSerializerParams;
var input_media_animation_serializer_1 = require("./serialize/input_media_animation_serializer");
exports.InputMediaAnimationSerializer = input_media_animation_serializer_1.InputMediaAnimationSerializer;
exports.InputMediaAnimationSerializerParams = input_media_animation_serializer_1.InputMediaAnimationSerializerParams;
var input_media_audio_serializer_1 = require("./serialize/input_media_audio_serializer");
exports.InputMediaAudioSerializer = input_media_audio_serializer_1.InputMediaAudioSerializer;
exports.InputMediaAudioSerializerParams = input_media_audio_serializer_1.InputMediaAudioSerializerParams;
var input_media_document_serializer_1 = require("./serialize/input_media_document_serializer");
exports.InputMediaDocumentSerializer = input_media_document_serializer_1.InputMediaDocumentSerializer;
exports.InputMediaDocumentSerializerParams = input_media_document_serializer_1.InputMediaDocumentSerializerParams;
var input_media_photo_serializer_1 = require("./serialize/input_media_photo_serializer");
exports.InputMediaPhotoSerializer = input_media_photo_serializer_1.InputMediaPhotoSerializer;
exports.InputMediaPhotoSerializerParams = input_media_photo_serializer_1.InputMediaPhotoSerializerParams;
var input_media_serializer_1 = require("./serialize/input_media_serializer");
exports.InputMediaSerializer = input_media_serializer_1.InputMediaSerializer;
exports.InputMediaSerializerParams = input_media_serializer_1.InputMediaSerializerParams;
var input_media_video_serializer_1 = require("./serialize/input_media_video_serializer");
exports.InputMediaVideoSerializer = input_media_video_serializer_1.InputMediaVideoSerializer;
exports.InputMediaVideoSerializerParams = input_media_video_serializer_1.InputMediaVideoSerializerParams;
var input_message_content_serializer_1 = require("./serialize/input_message_content_serializer");
exports.InputMessageContentSerializer = input_message_content_serializer_1.InputMessageContentSerializer;
exports.InputMessageContentSerializerParams = input_message_content_serializer_1.InputMessageContentSerializerParams;
var input_text_message_content_serializer_1 = require("./serialize/input_text_message_content_serializer");
exports.InputTextMessageContentSerializer = input_text_message_content_serializer_1.InputTextMessageContentSerializer;
exports.InputTextMessageContentSerializerParams = input_text_message_content_serializer_1.InputTextMessageContentSerializerParams;
var input_venue_message_content_serializer_1 = require("./serialize/input_venue_message_content_serializer");
exports.InputVenueMessageContentSerializer = input_venue_message_content_serializer_1.InputVenueMessageContentSerializer;
exports.InputVenueMessageContentSerializerParams = input_venue_message_content_serializer_1.InputVenueMessageContentSerializerParams;
var invoice_serializer_1 = require("./serialize/invoice_serializer");
exports.InvoiceSerializer = invoice_serializer_1.InvoiceSerializer;
exports.InvoiceSerializerParams = invoice_serializer_1.InvoiceSerializerParams;
var keyboard_button_poll_type_serializer_1 = require("./serialize/keyboard_button_poll_type_serializer");
exports.KeyboardButtonPollTypeSerializer = keyboard_button_poll_type_serializer_1.KeyboardButtonPollTypeSerializer;
exports.KeyboardButtonPollTypeSerializerParams = keyboard_button_poll_type_serializer_1.KeyboardButtonPollTypeSerializerParams;
var keyboard_button_serializer_1 = require("./serialize/keyboard_button_serializer");
exports.KeyboardButtonSerializer = keyboard_button_serializer_1.KeyboardButtonSerializer;
exports.KeyboardButtonSerializerParams = keyboard_button_serializer_1.KeyboardButtonSerializerParams;
var kick_chat_member_param_serializer_1 = require("./serialize/kick_chat_member_param_serializer");
exports.KickChatMemberParamSerializer = kick_chat_member_param_serializer_1.KickChatMemberParamSerializer;
exports.KickChatMemberParamSerializerParams = kick_chat_member_param_serializer_1.KickChatMemberParamSerializerParams;
var labeled_price_serializer_1 = require("./serialize/labeled_price_serializer");
exports.LabeledPriceSerializer = labeled_price_serializer_1.LabeledPriceSerializer;
exports.LabeledPriceSerializerParams = labeled_price_serializer_1.LabeledPriceSerializerParams;
var leave_chat_param_serializer_1 = require("./serialize/leave_chat_param_serializer");
exports.LeaveChatParamSerializer = leave_chat_param_serializer_1.LeaveChatParamSerializer;
exports.LeaveChatParamSerializerParams = leave_chat_param_serializer_1.LeaveChatParamSerializerParams;
var location_serializer_1 = require("./serialize/location_serializer");
exports.LocationSerializer = location_serializer_1.LocationSerializer;
exports.LocationSerializerParams = location_serializer_1.LocationSerializerParams;
var login_url_serializer_1 = require("./serialize/login_url_serializer");
exports.LoginUrlSerializer = login_url_serializer_1.LoginUrlSerializer;
exports.LoginUrlSerializerParams = login_url_serializer_1.LoginUrlSerializerParams;
var mask_position_serializer_1 = require("./serialize/mask_position_serializer");
exports.MaskPositionSerializer = mask_position_serializer_1.MaskPositionSerializer;
exports.MaskPositionSerializerParams = mask_position_serializer_1.MaskPositionSerializerParams;
var message_entity_serializer_1 = require("./serialize/message_entity_serializer");
exports.MessageEntitySerializer = message_entity_serializer_1.MessageEntitySerializer;
exports.MessageEntitySerializerParams = message_entity_serializer_1.MessageEntitySerializerParams;
var message_serializer_1 = require("./serialize/message_serializer");
exports.MessageSerializer = message_serializer_1.MessageSerializer;
exports.MessageSerializerParams = message_serializer_1.MessageSerializerParams;
var order_info_serializer_1 = require("./serialize/order_info_serializer");
exports.OrderInfoSerializer = order_info_serializer_1.OrderInfoSerializer;
exports.OrderInfoSerializerParams = order_info_serializer_1.OrderInfoSerializerParams;
var passport_data_serializer_1 = require("./serialize/passport_data_serializer");
exports.PassportDataSerializer = passport_data_serializer_1.PassportDataSerializer;
exports.PassportDataSerializerParams = passport_data_serializer_1.PassportDataSerializerParams;
var passport_element_error_data_field_serializer_1 = require("./serialize/passport_element_error_data_field_serializer");
exports.PassportElementErrorDataFieldSerializer = passport_element_error_data_field_serializer_1.PassportElementErrorDataFieldSerializer;
exports.PassportElementErrorDataFieldSerializerParams = passport_element_error_data_field_serializer_1.PassportElementErrorDataFieldSerializerParams;
var passport_element_error_files_serializer_1 = require("./serialize/passport_element_error_files_serializer");
exports.PassportElementErrorFilesSerializer = passport_element_error_files_serializer_1.PassportElementErrorFilesSerializer;
exports.PassportElementErrorFilesSerializerParams = passport_element_error_files_serializer_1.PassportElementErrorFilesSerializerParams;
var passport_element_error_file_serializer_1 = require("./serialize/passport_element_error_file_serializer");
exports.PassportElementErrorFileSerializer = passport_element_error_file_serializer_1.PassportElementErrorFileSerializer;
exports.PassportElementErrorFileSerializerParams = passport_element_error_file_serializer_1.PassportElementErrorFileSerializerParams;
var passport_element_error_front_side_serializer_1 = require("./serialize/passport_element_error_front_side_serializer");
exports.PassportElementErrorFrontSideSerializer = passport_element_error_front_side_serializer_1.PassportElementErrorFrontSideSerializer;
exports.PassportElementErrorFrontSideSerializerParams = passport_element_error_front_side_serializer_1.PassportElementErrorFrontSideSerializerParams;
var passport_element_error_reverse_side_serializer_1 = require("./serialize/passport_element_error_reverse_side_serializer");
exports.PassportElementErrorReverseSideSerializer = passport_element_error_reverse_side_serializer_1.PassportElementErrorReverseSideSerializer;
exports.PassportElementErrorReverseSideSerializerParams = passport_element_error_reverse_side_serializer_1.PassportElementErrorReverseSideSerializerParams;
var passport_element_error_selfie_serializer_1 = require("./serialize/passport_element_error_selfie_serializer");
exports.PassportElementErrorSelfieSerializer = passport_element_error_selfie_serializer_1.PassportElementErrorSelfieSerializer;
exports.PassportElementErrorSelfieSerializerParams = passport_element_error_selfie_serializer_1.PassportElementErrorSelfieSerializerParams;
var passport_element_error_serializer_1 = require("./serialize/passport_element_error_serializer");
exports.PassportElementErrorSerializer = passport_element_error_serializer_1.PassportElementErrorSerializer;
exports.PassportElementErrorSerializerParams = passport_element_error_serializer_1.PassportElementErrorSerializerParams;
var passport_element_error_translation_files_serializer_1 = require("./serialize/passport_element_error_translation_files_serializer");
exports.PassportElementErrorTranslationFilesSerializer = passport_element_error_translation_files_serializer_1.PassportElementErrorTranslationFilesSerializer;
exports.PassportElementErrorTranslationFilesSerializerParams = passport_element_error_translation_files_serializer_1.PassportElementErrorTranslationFilesSerializerParams;
var passport_element_error_translation_file_serializer_1 = require("./serialize/passport_element_error_translation_file_serializer");
exports.PassportElementErrorTranslationFileSerializer = passport_element_error_translation_file_serializer_1.PassportElementErrorTranslationFileSerializer;
exports.PassportElementErrorTranslationFileSerializerParams = passport_element_error_translation_file_serializer_1.PassportElementErrorTranslationFileSerializerParams;
var passport_element_error_unspecified_serializer_1 = require("./serialize/passport_element_error_unspecified_serializer");
exports.PassportElementErrorUnspecifiedSerializer = passport_element_error_unspecified_serializer_1.PassportElementErrorUnspecifiedSerializer;
exports.PassportElementErrorUnspecifiedSerializerParams = passport_element_error_unspecified_serializer_1.PassportElementErrorUnspecifiedSerializerParams;
var passport_file_serializer_1 = require("./serialize/passport_file_serializer");
exports.PassportFileSerializer = passport_file_serializer_1.PassportFileSerializer;
exports.PassportFileSerializerParams = passport_file_serializer_1.PassportFileSerializerParams;
var photo_size_serializer_1 = require("./serialize/photo_size_serializer");
exports.PhotoSizeSerializer = photo_size_serializer_1.PhotoSizeSerializer;
exports.PhotoSizeSerializerParams = photo_size_serializer_1.PhotoSizeSerializerParams;
var pin_chat_message_param_serializer_1 = require("./serialize/pin_chat_message_param_serializer");
exports.PinChatMessageParamSerializer = pin_chat_message_param_serializer_1.PinChatMessageParamSerializer;
exports.PinChatMessageParamSerializerParams = pin_chat_message_param_serializer_1.PinChatMessageParamSerializerParams;
var poll_answer_serializer_1 = require("./serialize/poll_answer_serializer");
exports.PollAnswerSerializer = poll_answer_serializer_1.PollAnswerSerializer;
exports.PollAnswerSerializerParams = poll_answer_serializer_1.PollAnswerSerializerParams;
var poll_option_serializer_1 = require("./serialize/poll_option_serializer");
exports.PollOptionSerializer = poll_option_serializer_1.PollOptionSerializer;
exports.PollOptionSerializerParams = poll_option_serializer_1.PollOptionSerializerParams;
var poll_serializer_1 = require("./serialize/poll_serializer");
exports.PollSerializer = poll_serializer_1.PollSerializer;
exports.PollSerializerParams = poll_serializer_1.PollSerializerParams;
var pre_checkout_query_serializer_1 = require("./serialize/pre_checkout_query_serializer");
exports.PreCheckoutQuerySerializer = pre_checkout_query_serializer_1.PreCheckoutQuerySerializer;
exports.PreCheckoutQuerySerializerParams = pre_checkout_query_serializer_1.PreCheckoutQuerySerializerParams;
var promote_chat_member_param_serializer_1 = require("./serialize/promote_chat_member_param_serializer");
exports.PromoteChatMemberParamSerializer = promote_chat_member_param_serializer_1.PromoteChatMemberParamSerializer;
exports.PromoteChatMemberParamSerializerParams = promote_chat_member_param_serializer_1.PromoteChatMemberParamSerializerParams;
var reply_keyboard_markup_serializer_1 = require("./serialize/reply_keyboard_markup_serializer");
exports.ReplyKeyboardMarkupSerializer = reply_keyboard_markup_serializer_1.ReplyKeyboardMarkupSerializer;
exports.ReplyKeyboardMarkupSerializerParams = reply_keyboard_markup_serializer_1.ReplyKeyboardMarkupSerializerParams;
var reply_keyboard_remove_serializer_1 = require("./serialize/reply_keyboard_remove_serializer");
exports.ReplyKeyboardRemoveSerializer = reply_keyboard_remove_serializer_1.ReplyKeyboardRemoveSerializer;
exports.ReplyKeyboardRemoveSerializerParams = reply_keyboard_remove_serializer_1.ReplyKeyboardRemoveSerializerParams;
var response_parameters_serializer_1 = require("./serialize/response_parameters_serializer");
exports.ResponseParametersSerializer = response_parameters_serializer_1.ResponseParametersSerializer;
exports.ResponseParametersSerializerParams = response_parameters_serializer_1.ResponseParametersSerializerParams;
var restrict_chat_member_param_serializer_1 = require("./serialize/restrict_chat_member_param_serializer");
exports.RestrictChatMemberParamSerializer = restrict_chat_member_param_serializer_1.RestrictChatMemberParamSerializer;
exports.RestrictChatMemberParamSerializerParams = restrict_chat_member_param_serializer_1.RestrictChatMemberParamSerializerParams;
var send_animation_param_serializer_1 = require("./serialize/send_animation_param_serializer");
exports.SendAnimationParamSerializer = send_animation_param_serializer_1.SendAnimationParamSerializer;
exports.SendAnimationParamSerializerParams = send_animation_param_serializer_1.SendAnimationParamSerializerParams;
var send_audio_param_serializer_1 = require("./serialize/send_audio_param_serializer");
exports.SendAudioParamSerializer = send_audio_param_serializer_1.SendAudioParamSerializer;
exports.SendAudioParamSerializerParams = send_audio_param_serializer_1.SendAudioParamSerializerParams;
var send_chat_action_param_serializer_1 = require("./serialize/send_chat_action_param_serializer");
exports.SendChatActionParamSerializer = send_chat_action_param_serializer_1.SendChatActionParamSerializer;
exports.SendChatActionParamSerializerParams = send_chat_action_param_serializer_1.SendChatActionParamSerializerParams;
var send_contact_param_serializer_1 = require("./serialize/send_contact_param_serializer");
exports.SendContactParamSerializer = send_contact_param_serializer_1.SendContactParamSerializer;
exports.SendContactParamSerializerParams = send_contact_param_serializer_1.SendContactParamSerializerParams;
var send_dice_param_serializer_1 = require("./serialize/send_dice_param_serializer");
exports.SendDiceParamSerializer = send_dice_param_serializer_1.SendDiceParamSerializer;
exports.SendDiceParamSerializerParams = send_dice_param_serializer_1.SendDiceParamSerializerParams;
var send_document_param_serializer_1 = require("./serialize/send_document_param_serializer");
exports.SendDocumentParamSerializer = send_document_param_serializer_1.SendDocumentParamSerializer;
exports.SendDocumentParamSerializerParams = send_document_param_serializer_1.SendDocumentParamSerializerParams;
var send_game_param_serializer_1 = require("./serialize/send_game_param_serializer");
exports.SendGameParamSerializer = send_game_param_serializer_1.SendGameParamSerializer;
exports.SendGameParamSerializerParams = send_game_param_serializer_1.SendGameParamSerializerParams;
var send_invoice_param_serializer_1 = require("./serialize/send_invoice_param_serializer");
exports.SendInvoiceParamSerializer = send_invoice_param_serializer_1.SendInvoiceParamSerializer;
exports.SendInvoiceParamSerializerParams = send_invoice_param_serializer_1.SendInvoiceParamSerializerParams;
var send_location_param_serializer_1 = require("./serialize/send_location_param_serializer");
exports.SendLocationParamSerializer = send_location_param_serializer_1.SendLocationParamSerializer;
exports.SendLocationParamSerializerParams = send_location_param_serializer_1.SendLocationParamSerializerParams;
var send_media_group_param_serializer_1 = require("./serialize/send_media_group_param_serializer");
exports.SendMediaGroupParamSerializer = send_media_group_param_serializer_1.SendMediaGroupParamSerializer;
exports.SendMediaGroupParamSerializerParams = send_media_group_param_serializer_1.SendMediaGroupParamSerializerParams;
var send_message_param_serializer_1 = require("./serialize/send_message_param_serializer");
exports.SendMessageParamSerializer = send_message_param_serializer_1.SendMessageParamSerializer;
exports.SendMessageParamSerializerParams = send_message_param_serializer_1.SendMessageParamSerializerParams;
var send_photo_param_serializer_1 = require("./serialize/send_photo_param_serializer");
exports.SendPhotoParamSerializer = send_photo_param_serializer_1.SendPhotoParamSerializer;
exports.SendPhotoParamSerializerParams = send_photo_param_serializer_1.SendPhotoParamSerializerParams;
var send_poll_param_serializer_1 = require("./serialize/send_poll_param_serializer");
exports.SendPollParamSerializer = send_poll_param_serializer_1.SendPollParamSerializer;
exports.SendPollParamSerializerParams = send_poll_param_serializer_1.SendPollParamSerializerParams;
var send_sticker_param_serializer_1 = require("./serialize/send_sticker_param_serializer");
exports.SendStickerParamSerializer = send_sticker_param_serializer_1.SendStickerParamSerializer;
exports.SendStickerParamSerializerParams = send_sticker_param_serializer_1.SendStickerParamSerializerParams;
var send_venue_param_serializer_1 = require("./serialize/send_venue_param_serializer");
exports.SendVenueParamSerializer = send_venue_param_serializer_1.SendVenueParamSerializer;
exports.SendVenueParamSerializerParams = send_venue_param_serializer_1.SendVenueParamSerializerParams;
var send_video_note_param_serializer_1 = require("./serialize/send_video_note_param_serializer");
exports.SendVideoNoteParamSerializer = send_video_note_param_serializer_1.SendVideoNoteParamSerializer;
exports.SendVideoNoteParamSerializerParams = send_video_note_param_serializer_1.SendVideoNoteParamSerializerParams;
var send_video_param_serializer_1 = require("./serialize/send_video_param_serializer");
exports.SendVideoParamSerializer = send_video_param_serializer_1.SendVideoParamSerializer;
exports.SendVideoParamSerializerParams = send_video_param_serializer_1.SendVideoParamSerializerParams;
var send_voice_param_serializer_1 = require("./serialize/send_voice_param_serializer");
exports.SendVoiceParamSerializer = send_voice_param_serializer_1.SendVoiceParamSerializer;
exports.SendVoiceParamSerializerParams = send_voice_param_serializer_1.SendVoiceParamSerializerParams;
var serializer_1 = require("./serialize/serializer");
exports.Serializer = serializer_1.Serializer;
var set_chat_administrator_custom_title_param_serializer_1 = require("./serialize/set_chat_administrator_custom_title_param_serializer");
exports.SetChatAdministratorCustomTitleParamSerializer = set_chat_administrator_custom_title_param_serializer_1.SetChatAdministratorCustomTitleParamSerializer;
exports.SetChatAdministratorCustomTitleParamSerializerParams = set_chat_administrator_custom_title_param_serializer_1.SetChatAdministratorCustomTitleParamSerializerParams;
var set_chat_description_param_serializer_1 = require("./serialize/set_chat_description_param_serializer");
exports.SetChatDescriptionParamSerializer = set_chat_description_param_serializer_1.SetChatDescriptionParamSerializer;
exports.SetChatDescriptionParamSerializerParams = set_chat_description_param_serializer_1.SetChatDescriptionParamSerializerParams;
var set_chat_permissions_param_serializer_1 = require("./serialize/set_chat_permissions_param_serializer");
exports.SetChatPermissionsParamSerializer = set_chat_permissions_param_serializer_1.SetChatPermissionsParamSerializer;
exports.SetChatPermissionsParamSerializerParams = set_chat_permissions_param_serializer_1.SetChatPermissionsParamSerializerParams;
var set_chat_photo_param_serializer_1 = require("./serialize/set_chat_photo_param_serializer");
exports.SetChatPhotoParamSerializer = set_chat_photo_param_serializer_1.SetChatPhotoParamSerializer;
exports.SetChatPhotoParamSerializerParams = set_chat_photo_param_serializer_1.SetChatPhotoParamSerializerParams;
var set_chat_sticker_set_param_serializer_1 = require("./serialize/set_chat_sticker_set_param_serializer");
exports.SetChatStickerSetParamSerializer = set_chat_sticker_set_param_serializer_1.SetChatStickerSetParamSerializer;
exports.SetChatStickerSetParamSerializerParams = set_chat_sticker_set_param_serializer_1.SetChatStickerSetParamSerializerParams;
var set_chat_title_param_serializer_1 = require("./serialize/set_chat_title_param_serializer");
exports.SetChatTitleParamSerializer = set_chat_title_param_serializer_1.SetChatTitleParamSerializer;
exports.SetChatTitleParamSerializerParams = set_chat_title_param_serializer_1.SetChatTitleParamSerializerParams;
var set_game_score_param_serializer_1 = require("./serialize/set_game_score_param_serializer");
exports.SetGameScoreParamSerializer = set_game_score_param_serializer_1.SetGameScoreParamSerializer;
exports.SetGameScoreParamSerializerParams = set_game_score_param_serializer_1.SetGameScoreParamSerializerParams;
var set_my_commands_param_serializer_1 = require("./serialize/set_my_commands_param_serializer");
exports.SetMyCommandsParamSerializer = set_my_commands_param_serializer_1.SetMyCommandsParamSerializer;
exports.SetMyCommandsParamSerializerParams = set_my_commands_param_serializer_1.SetMyCommandsParamSerializerParams;
var set_passport_data_errors_param_serializer_1 = require("./serialize/set_passport_data_errors_param_serializer");
exports.SetPassportDataErrorsParamSerializer = set_passport_data_errors_param_serializer_1.SetPassportDataErrorsParamSerializer;
exports.SetPassportDataErrorsParamSerializerParams = set_passport_data_errors_param_serializer_1.SetPassportDataErrorsParamSerializerParams;
var set_sticker_position_in_set_param_serializer_1 = require("./serialize/set_sticker_position_in_set_param_serializer");
exports.SetStickerPositionInSetParamSerializer = set_sticker_position_in_set_param_serializer_1.SetStickerPositionInSetParamSerializer;
exports.SetStickerPositionInSetParamSerializerParams = set_sticker_position_in_set_param_serializer_1.SetStickerPositionInSetParamSerializerParams;
var set_sticker_set_thumb_param_serializer_1 = require("./serialize/set_sticker_set_thumb_param_serializer");
exports.SetStickerSetThumbParamSerializer = set_sticker_set_thumb_param_serializer_1.SetStickerSetThumbParamSerializer;
exports.SetStickerSetThumbParamSerializerParams = set_sticker_set_thumb_param_serializer_1.SetStickerSetThumbParamSerializerParams;
var set_webhook_param_serializer_1 = require("./serialize/set_webhook_param_serializer");
exports.SetWebhookParamSerializer = set_webhook_param_serializer_1.SetWebhookParamSerializer;
exports.SetWebhookParamSerializerParams = set_webhook_param_serializer_1.SetWebhookParamSerializerParams;
var shipping_address_serializer_1 = require("./serialize/shipping_address_serializer");
exports.ShippingAddressSerializer = shipping_address_serializer_1.ShippingAddressSerializer;
exports.ShippingAddressSerializerParams = shipping_address_serializer_1.ShippingAddressSerializerParams;
var shipping_option_serializer_1 = require("./serialize/shipping_option_serializer");
exports.ShippingOptionSerializer = shipping_option_serializer_1.ShippingOptionSerializer;
exports.ShippingOptionSerializerParams = shipping_option_serializer_1.ShippingOptionSerializerParams;
var shipping_query_serializer_1 = require("./serialize/shipping_query_serializer");
exports.ShippingQuerySerializer = shipping_query_serializer_1.ShippingQuerySerializer;
exports.ShippingQuerySerializerParams = shipping_query_serializer_1.ShippingQuerySerializerParams;
var sticker_serializer_1 = require("./serialize/sticker_serializer");
exports.StickerSerializer = sticker_serializer_1.StickerSerializer;
exports.StickerSerializerParams = sticker_serializer_1.StickerSerializerParams;
var sticker_set_serializer_1 = require("./serialize/sticker_set_serializer");
exports.StickerSetSerializer = sticker_set_serializer_1.StickerSetSerializer;
exports.StickerSetSerializerParams = sticker_set_serializer_1.StickerSetSerializerParams;
var stop_message_live_location_param_serializer_1 = require("./serialize/stop_message_live_location_param_serializer");
exports.StopMessageLiveLocationParamSerializer = stop_message_live_location_param_serializer_1.StopMessageLiveLocationParamSerializer;
exports.StopMessageLiveLocationParamSerializerParams = stop_message_live_location_param_serializer_1.StopMessageLiveLocationParamSerializerParams;
var stop_poll_param_serializer_1 = require("./serialize/stop_poll_param_serializer");
exports.StopPollParamSerializer = stop_poll_param_serializer_1.StopPollParamSerializer;
exports.StopPollParamSerializerParams = stop_poll_param_serializer_1.StopPollParamSerializerParams;
var successful_payment_serializer_1 = require("./serialize/successful_payment_serializer");
exports.SuccessfulPaymentSerializer = successful_payment_serializer_1.SuccessfulPaymentSerializer;
exports.SuccessfulPaymentSerializerParams = successful_payment_serializer_1.SuccessfulPaymentSerializerParams;
var unban_chat_member_param_serializer_1 = require("./serialize/unban_chat_member_param_serializer");
exports.UnbanChatMemberParamSerializer = unban_chat_member_param_serializer_1.UnbanChatMemberParamSerializer;
exports.UnbanChatMemberParamSerializerParams = unban_chat_member_param_serializer_1.UnbanChatMemberParamSerializerParams;
var unpin_chat_message_param_serializer_1 = require("./serialize/unpin_chat_message_param_serializer");
exports.UnpinChatMessageParamSerializer = unpin_chat_message_param_serializer_1.UnpinChatMessageParamSerializer;
exports.UnpinChatMessageParamSerializerParams = unpin_chat_message_param_serializer_1.UnpinChatMessageParamSerializerParams;
var update_serializer_1 = require("./serialize/update_serializer");
exports.UpdateSerializer = update_serializer_1.UpdateSerializer;
exports.UpdateSerializerParams = update_serializer_1.UpdateSerializerParams;
var upload_sticker_file_param_serializer_1 = require("./serialize/upload_sticker_file_param_serializer");
exports.UploadStickerFileParamSerializer = upload_sticker_file_param_serializer_1.UploadStickerFileParamSerializer;
exports.UploadStickerFileParamSerializerParams = upload_sticker_file_param_serializer_1.UploadStickerFileParamSerializerParams;
var user_profile_photos_serializer_1 = require("./serialize/user_profile_photos_serializer");
exports.UserProfilePhotosSerializer = user_profile_photos_serializer_1.UserProfilePhotosSerializer;
exports.UserProfilePhotosSerializerParams = user_profile_photos_serializer_1.UserProfilePhotosSerializerParams;
var user_serializer_1 = require("./serialize/user_serializer");
exports.UserSerializer = user_serializer_1.UserSerializer;
exports.UserSerializerParams = user_serializer_1.UserSerializerParams;
var venue_serializer_1 = require("./serialize/venue_serializer");
exports.VenueSerializer = venue_serializer_1.VenueSerializer;
exports.VenueSerializerParams = venue_serializer_1.VenueSerializerParams;
var video_note_serializer_1 = require("./serialize/video_note_serializer");
exports.VideoNoteSerializer = video_note_serializer_1.VideoNoteSerializer;
exports.VideoNoteSerializerParams = video_note_serializer_1.VideoNoteSerializerParams;
var video_serializer_1 = require("./serialize/video_serializer");
exports.VideoSerializer = video_serializer_1.VideoSerializer;
exports.VideoSerializerParams = video_serializer_1.VideoSerializerParams;
var voice_serializer_1 = require("./serialize/voice_serializer");
exports.VoiceSerializer = voice_serializer_1.VoiceSerializer;
exports.VoiceSerializerParams = voice_serializer_1.VoiceSerializerParams;
var webhook_info_serializer_1 = require("./serialize/webhook_info_serializer");
exports.WebhookInfoSerializer = webhook_info_serializer_1.WebhookInfoSerializer;
exports.WebhookInfoSerializerParams = webhook_info_serializer_1.WebhookInfoSerializerParams;
