// telegram-bot-ts/build/src/features/tg_bot/presentation/mixins/telegram_bot_pubsub.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var telegram_publisher_1 = require("../pubsub/publishers/telegram_publisher");
var telegram_text_subscriber_1 = require("../pubsub/subscribers/telegram_text_subscriber");
var telegram_callback_subscriber_1 = require("../pubsub/subscribers/telegram_callback_subscriber");
var telegram_all_subscriber_1 = require("../pubsub/subscribers/telegram_all_subscriber");
var TelegramBotPubSub = /** @class */ (function () {
    function TelegramBotPubSub() {
    }
    TelegramBotPubSub.prototype.initPubSub = function () {
        this.telegramPublisher = new telegram_publisher_1.default();
    };
    Object.defineProperty(TelegramBotPubSub.prototype, "botTelegramPublisher", {
        get: function () {
            return this.telegramPublisher;
        },
        enumerable: true,
        configurable: true
    });
    TelegramBotPubSub.prototype.onText = function (callback) {
        var telegramTextSubscriber = new telegram_text_subscriber_1.default(callback);
        this.telegramPublisher.subscribe(telegramTextSubscriber);
    };
    TelegramBotPubSub.prototype.onCallback = function (callback) {
        var telegramCallbackSubscriber = new telegram_callback_subscriber_1.default(callback);
        this.telegramPublisher.subscribe(telegramCallbackSubscriber);
    };
    TelegramBotPubSub.prototype.onAll = function (callback) {
        var telegramAllSubscriber = new telegram_all_subscriber_1.default(callback);
        this.telegramPublisher.subscribe(telegramAllSubscriber);
    };
    return TelegramBotPubSub;
}());
exports.default = TelegramBotPubSub;
