// telegram-bot-ts/build/src/features/tg_bot/presentation/mixins/telegram_bot_methods.js
"use strict";
var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
    return new (P || (P = Promise))(function (resolve, reject) {
        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
        step((generator = generator.apply(thisArg, _arguments || [])).next());
    });
};
var __generator = (this && this.__generator) || function (thisArg, body) {
    var _ = { label: 0, sent: function() { if (t[0] & 1) throw t[1]; return t[1]; }, trys: [], ops: [] }, f, y, t, g;
    return g = { next: verb(0), "throw": verb(1), "return": verb(2) }, typeof Symbol === "function" && (g[Symbol.iterator] = function() { return this; }), g;
    function verb(n) { return function (v) { return step([n, v]); }; }
    function step(op) {
        if (f) throw new TypeError("Generator is already executing.");
        while (_) try {
            if (f = 1, y && (t = op[0] & 2 ? y["return"] : op[0] ? y["throw"] || ((t = y["return"]) && t.call(y), 0) : y.next) && !(t = t.call(y, op[1])).done) return t;
            if (y = 0, t) op = [op[0] & 2, t.value];
            switch (op[0]) {
                case 0: case 1: t = op; break;
                case 4: _.label++; return { value: op[1], done: false };
                case 5: _.label++; y = op[1]; op = [0]; continue;
                case 7: op = _.ops.pop(); _.trys.pop(); continue;
                default:
                    if (!(t = _.trys, t = t.length > 0 && t[t.length - 1]) && (op[0] === 6 || op[0] === 2)) { _ = 0; continue; }
                    if (op[0] === 3 && (!t || (op[1] > t[0] && op[1] < t[3]))) { _.label = op[1]; break; }
                    if (op[0] === 6 && _.label < t[1]) { _.label = t[1]; t = op; break; }
                    if (t && _.label < t[2]) { _.label = t[2]; _.ops.push(op); break; }
                    if (t[2]) _.ops.pop();
                    _.trys.pop(); continue;
            }
            op = body.call(thisArg, _);
        } catch (e) { op = [6, e]; y = 0; } finally { f = t = 0; }
        if (op[0] & 5) throw op[1]; return { value: op[0] ? op[1] : void 0, done: true };
    }
};
Object.defineProperty(exports, "__esModule", { value: true });
var send_photo_1 = require("../../domain/usecases/send_photo");
var send_message_1 = require("../../domain/usecases/send_message");
var send_animation_1 = require("../../domain/usecases/send_animation");
var send_video_1 = require("../../domain/usecases/send_video");
var edit_message_text_1 = require("../../domain/usecases/edit_message_text");
var edit_message_reply_markup_1 = require("../../domain/usecases/edit_message_reply_markup");
var get_me_1 = require("../../domain/usecases/get_me");
var telegram_bot_ts_types_1 = require("telegram-bot-ts-types");
var TelegramBotMethods = /** @class */ (function () {
    function TelegramBotMethods() {
    }
    TelegramBotMethods.prototype.initUseCases = function (telegramRepository) {
        this.getMeUseCase = new get_me_1.GetMe(telegramRepository);
        this.sendMessageUseCase = new send_message_1.SendMessage(telegramRepository);
        this.sendPhotoUseCase = new send_photo_1.SendPhoto(telegramRepository);
        this.sendAnimationUseCase = new send_animation_1.SendAnimation(telegramRepository);
        this.sendVideoUseCase = new send_video_1.SendVideo(telegramRepository);
        this.editMessageTextUseCase = new edit_message_text_1.EditMessageText(telegramRepository);
        this.editMessageReplyMarkupUseCase = new edit_message_reply_markup_1.EditMessageReplyMarkup(telegramRepository);
    };
    TelegramBotMethods.prototype.getMe = function () {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.getMeUseCase.execute()];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    TelegramBotMethods.prototype.sendMessage = function (message) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.sendMessageUseCase.execute(new telegram_bot_ts_types_1.SendMessageParam(message))];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    TelegramBotMethods.prototype.sendPhoto = function (message) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.sendPhotoUseCase.execute(new telegram_bot_ts_types_1.SendPhotoParam(message))];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    TelegramBotMethods.prototype.sendAnimation = function (message) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.sendAnimationUseCase.execute(new telegram_bot_ts_types_1.SendAnimationParam(message))];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    TelegramBotMethods.prototype.sendVideo = function (message) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.sendVideoUseCase.execute(new telegram_bot_ts_types_1.SendVideoParam(message))];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    TelegramBotMethods.prototype.editMessageText = function (message) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.editMessageTextUseCase.execute(new telegram_bot_ts_types_1.EditMessageTextParam(message))];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    TelegramBotMethods.prototype.editMessageReplyMarkup = function (message) {
        return __awaiter(this, void 0, void 0, function () {
            return __generator(this, function (_a) {
                switch (_a.label) {
                    case 0: return [4 /*yield*/, this.editMessageReplyMarkupUseCase.execute(new telegram_bot_ts_types_1.EditMessageReplyMarkupParam(message))];
                    case 1: return [2 /*return*/, _a.sent()];
                }
            });
        });
    };
    return TelegramBotMethods;
}());
exports.default = TelegramBotMethods;
