// telegram-bot-ts-types/build/serialize/add_sticker_to_set_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var add_sticker_to_set_param_1 = require("../method_params/add_sticker_to_set_param");
var mask_position_serializer_1 = require("./mask_position_serializer");
var _MaskPositionSerializer = mask_position_serializer_1.default;
var params = {
    userId: {
        required: true,
        type: 'number'
    },
    name: {
        required: true,
        type: 'string'
    },
    pngSticker: {
        required: true,
        type: 'InputFile | string'
    },
    tgsSticker: {
        required: false,
        type: 'InputFile'
    },
    emojis: {
        required: true,
        type: 'string'
    },
    maskPosition: {
        required: false,
        type: 'MaskPosition'
    }
};
exports.AddStickerToSetParamSerializerParams = params;
var AddStickerToSetParamSerializer = new serializer_1.Serializer(add_sticker_to_set_param_1.default, 'AddStickerToSetParam', params);
exports.AddStickerToSetParamSerializer = AddStickerToSetParamSerializer;
exports.default = AddStickerToSetParamSerializer;
