// telegram-bot-ts-types/build/serialize/animation_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var animation_1 = require("../entities/animation");
var photo_size_serializer_1 = require("./photo_size_serializer");
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var params = {
    fileId: {
        required: true,
        type: 'string'
    },
    fileUniqueId: {
        required: true,
        type: 'string'
    },
    width: {
        required: true,
        type: 'number'
    },
    height: {
        required: true,
        type: 'number'
    },
    duration: {
        required: true,
        type: 'number'
    },
    thumb: {
        required: false,
        type: 'PhotoSize'
    },
    fileName: {
        required: false,
        type: 'string'
    },
    mimeType: {
        required: false,
        type: 'string'
    },
    fileSize: {
        required: false,
        type: 'number'
    }
};
exports.AnimationSerializerParams = params;
var AnimationSerializer = new serializer_1.Serializer(animation_1.default, 'Animation', params);
exports.AnimationSerializer = AnimationSerializer;
exports.default = AnimationSerializer;
