// telegram-bot-ts-types/build/serialize/answer_shipping_query_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var answer_shipping_query_param_1 = require("../method_params/answer_shipping_query_param");
var shipping_option_serializer_1 = require("./shipping_option_serializer");
var _ShippingOptionSerializer = shipping_option_serializer_1.default;
var params = {
    shippingQueryId: {
        required: true,
        type: 'string'
    },
    ok: {
        required: true,
        type: 'boolean'
    },
    shippingOptions: {
        required: false,
        type: 'ShippingOption[]'
    },
    errorMessage: {
        required: false,
        type: 'string'
    }
};
exports.AnswerShippingQueryParamSerializerParams = params;
var AnswerShippingQueryParamSerializer = new serializer_1.Serializer(answer_shipping_query_param_1.default, 'AnswerShippingQueryParam', params);
exports.AnswerShippingQueryParamSerializer = AnswerShippingQueryParamSerializer;
exports.default = AnswerShippingQueryParamSerializer;
