// telegram-bot-ts-types/build/serialize/answer_inline_query_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var answer_inline_query_param_1 = require("../method_params/answer_inline_query_param");
var inline_query_result_serializer_1 = require("./inline_query_result_serializer");
var _InlineQueryResultSerializer = inline_query_result_serializer_1.default;
var params = {
    inlineQueryId: {
        required: true,
        type: 'string'
    },
    results: {
        required: true,
        type: 'InlineQueryResult[]'
    },
    cacheTime: {
        required: false,
        type: 'number'
    },
    isPersonal: {
        required: false,
        type: 'boolean'
    },
    nextOffset: {
        required: false,
        type: 'string'
    },
    switchPmText: {
        required: false,
        type: 'string'
    },
    switchPmParameter: {
        required: false,
        type: 'string'
    }
};
exports.AnswerInlineQueryParamSerializerParams = params;
var AnswerInlineQueryParamSerializer = new serializer_1.Serializer(answer_inline_query_param_1.default, 'AnswerInlineQueryParam', params);
exports.AnswerInlineQueryParamSerializer = AnswerInlineQueryParamSerializer;
exports.default = AnswerInlineQueryParamSerializer;
