// telegram-bot-ts-types/build/serialize/audio_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var audio_1 = require("../entities/audio");
var photo_size_serializer_1 = require("./photo_size_serializer");
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var params = {
    fileId: {
        required: true,
        type: 'string'
    },
    fileUniqueId: {
        required: true,
        type: 'string'
    },
    duration: {
        required: true,
        type: 'number'
    },
    performer: {
        required: false,
        type: 'string'
    },
    title: {
        required: false,
        type: 'string'
    },
    mimeType: {
        required: false,
        type: 'string'
    },
    fileSize: {
        required: false,
        type: 'number'
    },
    thumb: {
        required: false,
        type: 'PhotoSize'
    }
};
exports.AudioSerializerParams = params;
var AudioSerializer = new serializer_1.Serializer(audio_1.default, 'Audio', params);
exports.AudioSerializer = AudioSerializer;
exports.default = AudioSerializer;
