// telegram-bot-ts-types/build/serialize/callback_query_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var callback_query_1 = require("../entities/callback_query");
var user_serializer_1 = require("./user_serializer");
var message_serializer_1 = require("./message_serializer");
var _UserSerializer = user_serializer_1.default;
var _MessageSerializer = message_serializer_1.default;
var params = {
    id: {
        required: true,
        type: 'string'
    },
    from: {
        required: true,
        type: 'User'
    },
    message: {
        required: false,
        type: 'Message'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    chatInstance: {
        required: true,
        type: 'string'
    },
    data: {
        required: false,
        type: 'string'
    },
    gameShortName: {
        required: false,
        type: 'string'
    }
};
exports.CallbackQuerySerializerParams = params;
var CallbackQuerySerializer = new serializer_1.Serializer(callback_query_1.default, 'CallbackQuery', params);
exports.CallbackQuerySerializer = CallbackQuerySerializer;
exports.default = CallbackQuerySerializer;
