// telegram-bot-ts-types/build/serialize/chat_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var chat_1 = require("../entities/chat");
var chat_photo_serializer_1 = require("./chat_photo_serializer");
var message_serializer_1 = require("./message_serializer");
var chat_permissions_serializer_1 = require("./chat_permissions_serializer");
var _ChatPhotoSerializer = chat_photo_serializer_1.default;
var _MessageSerializer = message_serializer_1.default;
var _ChatPermissionsSerializer = chat_permissions_serializer_1.default;
var params = {
    id: {
        required: true,
        type: 'number'
    },
    type: {
        required: true,
        type: 'string'
    },
    title: {
        required: false,
        type: 'string'
    },
    username: {
        required: false,
        type: 'string'
    },
    firstName: {
        required: false,
        type: 'string'
    },
    lastName: {
        required: false,
        type: 'string'
    },
    photo: {
        required: false,
        type: 'ChatPhoto'
    },
    description: {
        required: false,
        type: 'string'
    },
    inviteLink: {
        required: false,
        type: 'string'
    },
    pinnedMessage: {
        required: false,
        type: 'Message'
    },
    permissions: {
        required: false,
        type: 'ChatPermissions'
    },
    slowModeDelay: {
        required: false,
        type: 'number'
    },
    stickerSetName: {
        required: false,
        type: 'string'
    },
    canSetStickerSet: {
        required: false,
        type: 'boolean'
    }
};
exports.ChatSerializerParams = params;
var ChatSerializer = new serializer_1.Serializer(chat_1.default, 'Chat', params);
exports.ChatSerializer = ChatSerializer;
exports.default = ChatSerializer;
