// telegram-bot-ts-types/build/serialize/chat_permissions_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var chat_permissions_1 = require("../entities/chat_permissions");
var params = {
    canSendMessages: {
        required: false,
        type: 'boolean'
    },
    canSendMediaMessages: {
        required: false,
        type: 'boolean'
    },
    canSendPolls: {
        required: false,
        type: 'boolean'
    },
    canSendOtherMessages: {
        required: false,
        type: 'boolean'
    },
    canAddWebPagePreviews: {
        required: false,
        type: 'boolean'
    },
    canChangeInfo: {
        required: false,
        type: 'boolean'
    },
    canInviteUsers: {
        required: false,
        type: 'boolean'
    },
    canPinMessages: {
        required: false,
        type: 'boolean'
    }
};
exports.ChatPermissionsSerializerParams = params;
var ChatPermissionsSerializer = new serializer_1.Serializer(chat_permissions_1.default, 'ChatPermissions', params);
exports.ChatPermissionsSerializer = ChatPermissionsSerializer;
exports.default = ChatPermissionsSerializer;
