// telegram-bot-ts-types/build/serialize/chat_member_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var chat_member_1 = require("../entities/chat_member");
var user_serializer_1 = require("./user_serializer");
var _UserSerializer = user_serializer_1.default;
var params = {
    user: {
        required: true,
        type: 'User'
    },
    status: {
        required: true,
        type: 'string'
    },
    customTitle: {
        required: false,
        type: 'string'
    },
    untilDate: {
        required: false,
        type: 'number'
    },
    canBeEdited: {
        required: false,
        type: 'boolean'
    },
    canPostMessages: {
        required: false,
        type: 'boolean'
    },
    canEditMessages: {
        required: false,
        type: 'boolean'
    },
    canDeleteMessages: {
        required: false,
        type: 'boolean'
    },
    canRestrictMembers: {
        required: false,
        type: 'boolean'
    },
    canPromoteMembers: {
        required: false,
        type: 'boolean'
    },
    canChangeInfo: {
        required: false,
        type: 'boolean'
    },
    canInviteUsers: {
        required: false,
        type: 'boolean'
    },
    canPinMessages: {
        required: false,
        type: 'boolean'
    },
    isMember: {
        required: false,
        type: 'boolean'
    },
    canSendMessages: {
        required: false,
        type: 'boolean'
    },
    canSendMediaMessages: {
        required: false,
        type: 'boolean'
    },
    canSendPolls: {
        required: false,
        type: 'boolean'
    },
    canSendOtherMessages: {
        required: false,
        type: 'boolean'
    },
    canAddWebPagePreviews: {
        required: false,
        type: 'boolean'
    }
};
exports.ChatMemberSerializerParams = params;
var ChatMemberSerializer = new serializer_1.Serializer(chat_member_1.default, 'ChatMember', params);
exports.ChatMemberSerializer = ChatMemberSerializer;
exports.default = ChatMemberSerializer;
