// telegram-bot-ts-types/build/serialize/chosen_inline_result_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var chosen_inline_result_1 = require("../entities/chosen_inline_result");
var user_serializer_1 = require("./user_serializer");
var location_serializer_1 = require("./location_serializer");
var _UserSerializer = user_serializer_1.default;
var _LocationSerializer = location_serializer_1.default;
var params = {
    resultId: {
        required: true,
        type: 'string'
    },
    from: {
        required: true,
        type: 'User'
    },
    location: {
        required: false,
        type: 'Location'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    query: {
        required: true,
        type: 'string'
    }
};
exports.ChosenInlineResultSerializerParams = params;
var ChosenInlineResultSerializer = new serializer_1.Serializer(chosen_inline_result_1.default, 'ChosenInlineResult', params);
exports.ChosenInlineResultSerializer = ChosenInlineResultSerializer;
exports.default = ChosenInlineResultSerializer;
