// telegram-bot-ts-types/build/serialize/create_new_sticker_set_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var create_new_sticker_set_param_1 = require("../method_params/create_new_sticker_set_param");
var mask_position_serializer_1 = require("./mask_position_serializer");
var _MaskPositionSerializer = mask_position_serializer_1.default;
var params = {
    userId: {
        required: true,
        type: 'number'
    },
    name: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    pngSticker: {
        required: false,
        type: 'InputFile | string'
    },
    tgsSticker: {
        required: false,
        type: 'InputFile'
    },
    emojis: {
        required: true,
        type: 'string'
    },
    containsMasks: {
        required: false,
        type: 'boolean'
    },
    maskPosition: {
        required: false,
        type: 'MaskPosition'
    }
};
exports.CreateNewStickerSetParamSerializerParams = params;
var CreateNewStickerSetParamSerializer = new serializer_1.Serializer(create_new_sticker_set_param_1.default, 'CreateNewStickerSetParam', params);
exports.CreateNewStickerSetParamSerializer = CreateNewStickerSetParamSerializer;
exports.default = CreateNewStickerSetParamSerializer;
