// telegram-bot-ts-types/build/serialize/edit_message_caption_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var edit_message_caption_param_1 = require("../method_params/edit_message_caption_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: false,
        type: 'number | string'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.EditMessageCaptionParamSerializerParams = params;
var EditMessageCaptionParamSerializer = new serializer_1.Serializer(edit_message_caption_param_1.default, 'EditMessageCaptionParam', params);
exports.EditMessageCaptionParamSerializer = EditMessageCaptionParamSerializer;
exports.default = EditMessageCaptionParamSerializer;
