// telegram-bot-ts-types/build/serialize/edit_message_text_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var edit_message_text_param_1 = require("../method_params/edit_message_text_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: false,
        type: 'number | string'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    text: {
        required: true,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    disableWebPagePreview: {
        required: false,
        type: 'boolean'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.EditMessageTextParamSerializerParams = params;
var EditMessageTextParamSerializer = new serializer_1.Serializer(edit_message_text_param_1.default, 'EditMessageTextParam', params);
exports.EditMessageTextParamSerializer = EditMessageTextParamSerializer;
exports.default = EditMessageTextParamSerializer;
