// telegram-bot-ts-types/build/serialize/edit_message_media_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var edit_message_media_param_1 = require("../method_params/edit_message_media_param");
var input_media_serializer_1 = require("./input_media_serializer");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InputMediaSerializer = input_media_serializer_1.default;
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: false,
        type: 'number | string'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    media: {
        required: true,
        type: 'InputMedia'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.EditMessageMediaParamSerializerParams = params;
var EditMessageMediaParamSerializer = new serializer_1.Serializer(edit_message_media_param_1.default, 'EditMessageMediaParam', params);
exports.EditMessageMediaParamSerializer = EditMessageMediaParamSerializer;
exports.default = EditMessageMediaParamSerializer;
