// telegram-bot-ts-types/build/serialize/edit_message_reply_markup_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var edit_message_reply_markup_param_1 = require("../method_params/edit_message_reply_markup_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: false,
        type: 'number | string'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.EditMessageReplyMarkupParamSerializerParams = params;
var EditMessageReplyMarkupParamSerializer = new serializer_1.Serializer(edit_message_reply_markup_param_1.default, 'EditMessageReplyMarkupParam', params);
exports.EditMessageReplyMarkupParamSerializer = EditMessageReplyMarkupParamSerializer;
exports.default = EditMessageReplyMarkupParamSerializer;
