// telegram-bot-ts-types/build/serialize/edit_message_live_location_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var edit_message_live_location_param_1 = require("../method_params/edit_message_live_location_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: false,
        type: 'number | string'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    latitude: {
        required: true,
        type: 'number'
    },
    longitude: {
        required: true,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.EditMessageLiveLocationParamSerializerParams = params;
var EditMessageLiveLocationParamSerializer = new serializer_1.Serializer(edit_message_live_location_param_1.default, 'EditMessageLiveLocationParam', params);
exports.EditMessageLiveLocationParamSerializer = EditMessageLiveLocationParamSerializer;
exports.default = EditMessageLiveLocationParamSerializer;
