// telegram-bot-ts-types/build/serialize/encrypted_passport_element_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var encrypted_passport_element_1 = require("../entities/encrypted_passport_element");
var passport_file_serializer_1 = require("./passport_file_serializer");
var _PassportFileSerializer = passport_file_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    data: {
        required: false,
        type: 'string'
    },
    phoneNumber: {
        required: false,
        type: 'string'
    },
    email: {
        required: false,
        type: 'string'
    },
    files: {
        required: false,
        type: 'PassportFile[]'
    },
    frontSide: {
        required: false,
        type: 'PassportFile'
    },
    reverseSide: {
        required: false,
        type: 'PassportFile'
    },
    selfie: {
        required: false,
        type: 'PassportFile'
    },
    translation: {
        required: false,
        type: 'PassportFile[]'
    },
    hash: {
        required: true,
        type: 'string'
    }
};
exports.EncryptedPassportElementSerializerParams = params;
var EncryptedPassportElementSerializer = new serializer_1.Serializer(encrypted_passport_element_1.default, 'EncryptedPassportElement', params);
exports.EncryptedPassportElementSerializer = EncryptedPassportElementSerializer;
exports.default = EncryptedPassportElementSerializer;
