// telegram-bot-ts-types/build/serialize/game_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var game_1 = require("../entities/game");
var photo_size_serializer_1 = require("./photo_size_serializer");
var message_entity_serializer_1 = require("./message_entity_serializer");
var animation_serializer_1 = require("./animation_serializer");
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var _MessageEntitySerializer = message_entity_serializer_1.default;
var _AnimationSerializer = animation_serializer_1.default;
var params = {
    title: {
        required: true,
        type: 'string'
    },
    description: {
        required: true,
        type: 'string'
    },
    photo: {
        required: true,
        type: 'PhotoSize[]'
    },
    text: {
        required: false,
        type: 'string'
    },
    textEntities: {
        required: false,
        type: 'MessageEntity[]'
    },
    animation: {
        required: false,
        type: 'Animation'
    }
};
exports.GameSerializerParams = params;
var GameSerializer = new serializer_1.Serializer(game_1.default, 'Game', params);
exports.GameSerializer = GameSerializer;
exports.default = GameSerializer;
