// telegram-bot-ts-types/build/serialize/inline_query_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_1 = require("../entities/inline_query");
var user_serializer_1 = require("./user_serializer");
var location_serializer_1 = require("./location_serializer");
var _UserSerializer = user_serializer_1.default;
var _LocationSerializer = location_serializer_1.default;
var params = {
    id: {
        required: true,
        type: 'string'
    },
    from: {
        required: true,
        type: 'User'
    },
    location: {
        required: false,
        type: 'Location'
    },
    query: {
        required: true,
        type: 'string'
    },
    offset: {
        required: true,
        type: 'string'
    }
};
exports.InlineQuerySerializerParams = params;
var InlineQuerySerializer = new serializer_1.Serializer(inline_query_1.default, 'InlineQuery', params);
exports.InlineQuerySerializer = InlineQuerySerializer;
exports.default = InlineQuerySerializer;
