// telegram-bot-ts-types/build/serialize/inline_keyboard_button_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_keyboard_button_1 = require("../entities/inline_keyboard_button");
var login_url_serializer_1 = require("./login_url_serializer");
var callback_game_serializer_1 = require("./callback_game_serializer");
var _LoginUrlSerializer = login_url_serializer_1.default;
var _CallbackGameSerializer = callback_game_serializer_1.default;
var params = {
    text: {
        required: true,
        type: 'string'
    },
    url: {
        required: false,
        type: 'string'
    },
    loginUrl: {
        required: false,
        type: 'LoginUrl'
    },
    callbackData: {
        required: false,
        type: 'string'
    },
    switchInlineQuery: {
        required: false,
        type: 'string'
    },
    switchInlineQueryCurrentChat: {
        required: false,
        type: 'string'
    },
    callbackGame: {
        required: false,
        type: 'CallbackGame'
    },
    pay: {
        required: false,
        type: 'boolean'
    }
};
exports.InlineKeyboardButtonSerializerParams = params;
var InlineKeyboardButtonSerializer = new serializer_1.Serializer(inline_keyboard_button_1.default, 'InlineKeyboardButton', params);
exports.InlineKeyboardButtonSerializer = InlineKeyboardButtonSerializer;
exports.default = InlineKeyboardButtonSerializer;
