// telegram-bot-ts-types/build/serialize/inline_query_result_contact_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_contact_1 = require("../entities/inline_query_result_contact");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    phoneNumber: {
        required: true,
        type: 'string'
    },
    firstName: {
        required: true,
        type: 'string'
    },
    lastName: {
        required: false,
        type: 'string'
    },
    vcard: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    },
    thumbUrl: {
        required: false,
        type: 'string'
    },
    thumbWidth: {
        required: false,
        type: 'number'
    },
    thumbHeight: {
        required: false,
        type: 'number'
    }
};
exports.InlineQueryResultContactSerializerParams = params;
var InlineQueryResultContactSerializer = new serializer_1.Serializer(inline_query_result_contact_1.default, 'InlineQueryResultContact', params);
exports.InlineQueryResultContactSerializer = InlineQueryResultContactSerializer;
exports.default = InlineQueryResultContactSerializer;
