// telegram-bot-ts-types/build/serialize/inline_query_result_cached_mpeg4_gif_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_cached_mpeg4_gif_1 = require("../entities/inline_query_result_cached_mpeg4_gif");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    mpeg4FileId: {
        required: true,
        type: 'string'
    },
    title: {
        required: false,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultCachedMpeg4GifSerializerParams = params;
var InlineQueryResultCachedMpeg4GifSerializer = new serializer_1.Serializer(inline_query_result_cached_mpeg4_gif_1.default, 'InlineQueryResultCachedMpeg4Gif', params);
exports.InlineQueryResultCachedMpeg4GifSerializer = InlineQueryResultCachedMpeg4GifSerializer;
exports.default = InlineQueryResultCachedMpeg4GifSerializer;
