// telegram-bot-ts-types/build/serialize/inline_query_result_cached_document_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_cached_document_1 = require("../entities/inline_query_result_cached_document");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    documentFileId: {
        required: true,
        type: 'string'
    },
    description: {
        required: false,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultCachedDocumentSerializerParams = params;
var InlineQueryResultCachedDocumentSerializer = new serializer_1.Serializer(inline_query_result_cached_document_1.default, 'InlineQueryResultCachedDocument', params);
exports.InlineQueryResultCachedDocumentSerializer = InlineQueryResultCachedDocumentSerializer;
exports.default = InlineQueryResultCachedDocumentSerializer;
