// telegram-bot-ts-types/build/serialize/inline_query_result_article_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_article_1 = require("../entities/inline_query_result_article");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    inputMessageContent: {
        required: true,
        type: 'InputMessageContent'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    url: {
        required: false,
        type: 'string'
    },
    hideUrl: {
        required: false,
        type: 'boolean'
    },
    description: {
        required: false,
        type: 'string'
    },
    thumbUrl: {
        required: false,
        type: 'string'
    },
    thumbWidth: {
        required: false,
        type: 'number'
    },
    thumbHeight: {
        required: false,
        type: 'number'
    }
};
exports.InlineQueryResultArticleSerializerParams = params;
var InlineQueryResultArticleSerializer = new serializer_1.Serializer(inline_query_result_article_1.default, 'InlineQueryResultArticle', params);
exports.InlineQueryResultArticleSerializer = InlineQueryResultArticleSerializer;
exports.default = InlineQueryResultArticleSerializer;
