// telegram-bot-ts-types/build/serialize/inline_query_result_location_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_location_1 = require("../entities/inline_query_result_location");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    latitude: {
        required: true,
        type: 'number'
    },
    longitude: {
        required: true,
        type: 'number'
    },
    title: {
        required: true,
        type: 'string'
    },
    livePeriod: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    },
    thumbUrl: {
        required: false,
        type: 'string'
    },
    thumbWidth: {
        required: false,
        type: 'number'
    },
    thumbHeight: {
        required: false,
        type: 'number'
    }
};
exports.InlineQueryResultLocationSerializerParams = params;
var InlineQueryResultLocationSerializer = new serializer_1.Serializer(inline_query_result_location_1.default, 'InlineQueryResultLocation', params);
exports.InlineQueryResultLocationSerializer = InlineQueryResultLocationSerializer;
exports.default = InlineQueryResultLocationSerializer;
