// telegram-bot-ts-types/build/serialize/inline_query_result_voice_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_voice_1 = require("../entities/inline_query_result_voice");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    voiceUrl: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    voiceDuration: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultVoiceSerializerParams = params;
var InlineQueryResultVoiceSerializer = new serializer_1.Serializer(inline_query_result_voice_1.default, 'InlineQueryResultVoice', params);
exports.InlineQueryResultVoiceSerializer = InlineQueryResultVoiceSerializer;
exports.default = InlineQueryResultVoiceSerializer;
