// telegram-bot-ts-types/build/serialize/inline_query_result_video_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_video_1 = require("../entities/inline_query_result_video");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    videoUrl: {
        required: true,
        type: 'string'
    },
    mimeType: {
        required: true,
        type: 'string'
    },
    thumbUrl: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    videoWidth: {
        required: false,
        type: 'number'
    },
    videoHeight: {
        required: false,
        type: 'number'
    },
    videoDuration: {
        required: false,
        type: 'number'
    },
    description: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultVideoSerializerParams = params;
var InlineQueryResultVideoSerializer = new serializer_1.Serializer(inline_query_result_video_1.default, 'InlineQueryResultVideo', params);
exports.InlineQueryResultVideoSerializer = InlineQueryResultVideoSerializer;
exports.default = InlineQueryResultVideoSerializer;
