// telegram-bot-ts-types/build/serialize/inline_query_result_game_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_game_1 = require("../entities/inline_query_result_game");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    gameShortName: {
        required: true,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.InlineQueryResultGameSerializerParams = params;
var InlineQueryResultGameSerializer = new serializer_1.Serializer(inline_query_result_game_1.default, 'InlineQueryResultGame', params);
exports.InlineQueryResultGameSerializer = InlineQueryResultGameSerializer;
exports.default = InlineQueryResultGameSerializer;
