// telegram-bot-ts-types/build/serialize/inline_query_result_gif_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_gif_1 = require("../entities/inline_query_result_gif");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    gifUrl: {
        required: true,
        type: 'string'
    },
    gifWidth: {
        required: false,
        type: 'number'
    },
    gifHeight: {
        required: false,
        type: 'number'
    },
    gifDuration: {
        required: false,
        type: 'number'
    },
    thumbUrl: {
        required: true,
        type: 'string'
    },
    title: {
        required: false,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultGifSerializerParams = params;
var InlineQueryResultGifSerializer = new serializer_1.Serializer(inline_query_result_gif_1.default, 'InlineQueryResultGif', params);
exports.InlineQueryResultGifSerializer = InlineQueryResultGifSerializer;
exports.default = InlineQueryResultGifSerializer;
