// telegram-bot-ts-types/build/serialize/inline_query_result_cached_sticker_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_cached_sticker_1 = require("../entities/inline_query_result_cached_sticker");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    stickerFileId: {
        required: true,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultCachedStickerSerializerParams = params;
var InlineQueryResultCachedStickerSerializer = new serializer_1.Serializer(inline_query_result_cached_sticker_1.default, 'InlineQueryResultCachedSticker', params);
exports.InlineQueryResultCachedStickerSerializer = InlineQueryResultCachedStickerSerializer;
exports.default = InlineQueryResultCachedStickerSerializer;
