// telegram-bot-ts-types/build/serialize/inline_query_result_audio_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var inline_query_result_audio_1 = require("../entities/inline_query_result_audio");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var input_message_content_serializer_1 = require("./input_message_content_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _InputMessageContentSerializer = input_message_content_serializer_1.default;
var params = {
    type: {
        required: true,
        type: 'string'
    },
    id: {
        required: true,
        type: 'string'
    },
    audioUrl: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    performer: {
        required: false,
        type: 'string'
    },
    audioDuration: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    },
    inputMessageContent: {
        required: false,
        type: 'InputMessageContent'
    }
};
exports.InlineQueryResultAudioSerializerParams = params;
var InlineQueryResultAudioSerializer = new serializer_1.Serializer(inline_query_result_audio_1.default, 'InlineQueryResultAudio', params);
exports.InlineQueryResultAudioSerializer = InlineQueryResultAudioSerializer;
exports.default = InlineQueryResultAudioSerializer;
