// telegram-bot-ts-types/build/serialize/input_media_audio_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var input_media_audio_1 = require("../entities/input_media_audio");
var params = {
    type: {
        required: true,
        type: 'string'
    },
    media: {
        required: true,
        type: 'string'
    },
    thumb: {
        required: false,
        type: 'InputFile | string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    duration: {
        required: false,
        type: 'number'
    },
    performer: {
        required: false,
        type: 'string'
    },
    title: {
        required: false,
        type: 'string'
    }
};
exports.InputMediaAudioSerializerParams = params;
var InputMediaAudioSerializer = new serializer_1.Serializer(input_media_audio_1.default, 'InputMediaAudio', params);
exports.InputMediaAudioSerializer = InputMediaAudioSerializer;
exports.default = InputMediaAudioSerializer;
