// telegram-bot-ts-types/build/serialize/input_media_animation_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var input_media_animation_1 = require("../entities/input_media_animation");
var params = {
    type: {
        required: true,
        type: 'string'
    },
    media: {
        required: true,
        type: 'string'
    },
    thumb: {
        required: false,
        type: 'InputFile | string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    width: {
        required: false,
        type: 'number'
    },
    height: {
        required: false,
        type: 'number'
    },
    duration: {
        required: false,
        type: 'number'
    }
};
exports.InputMediaAnimationSerializerParams = params;
var InputMediaAnimationSerializer = new serializer_1.Serializer(input_media_animation_1.default, 'InputMediaAnimation', params);
exports.InputMediaAnimationSerializer = InputMediaAnimationSerializer;
exports.default = InputMediaAnimationSerializer;
