// telegram-bot-ts-types/build/serialize/input_media_video_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var input_media_video_1 = require("../entities/input_media_video");
var params = {
    type: {
        required: true,
        type: 'string'
    },
    media: {
        required: true,
        type: 'string'
    },
    thumb: {
        required: false,
        type: 'InputFile | string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    width: {
        required: false,
        type: 'number'
    },
    height: {
        required: false,
        type: 'number'
    },
    duration: {
        required: false,
        type: 'number'
    },
    supportsStreaming: {
        required: false,
        type: 'boolean'
    }
};
exports.InputMediaVideoSerializerParams = params;
var InputMediaVideoSerializer = new serializer_1.Serializer(input_media_video_1.default, 'InputMediaVideo', params);
exports.InputMediaVideoSerializer = InputMediaVideoSerializer;
exports.default = InputMediaVideoSerializer;
