// telegram-bot-ts-types/build/serialize/message_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var message_1 = require("../entities/message");
var user_serializer_1 = require("./user_serializer");
var chat_serializer_1 = require("./chat_serializer");
var message_entity_serializer_1 = require("./message_entity_serializer");
var audio_serializer_1 = require("./audio_serializer");
var document_serializer_1 = require("./document_serializer");
var animation_serializer_1 = require("./animation_serializer");
var game_serializer_1 = require("./game_serializer");
var photo_size_serializer_1 = require("./photo_size_serializer");
var sticker_serializer_1 = require("./sticker_serializer");
var video_serializer_1 = require("./video_serializer");
var voice_serializer_1 = require("./voice_serializer");
var video_note_serializer_1 = require("./video_note_serializer");
var contact_serializer_1 = require("./contact_serializer");
var location_serializer_1 = require("./location_serializer");
var venue_serializer_1 = require("./venue_serializer");
var poll_serializer_1 = require("./poll_serializer");
var dice_serializer_1 = require("./dice_serializer");
var invoice_serializer_1 = require("./invoice_serializer");
var successful_payment_serializer_1 = require("./successful_payment_serializer");
var passport_data_serializer_1 = require("./passport_data_serializer");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _UserSerializer = user_serializer_1.default;
var _ChatSerializer = chat_serializer_1.default;
var _MessageEntitySerializer = message_entity_serializer_1.default;
var _AudioSerializer = audio_serializer_1.default;
var _DocumentSerializer = document_serializer_1.default;
var _AnimationSerializer = animation_serializer_1.default;
var _GameSerializer = game_serializer_1.default;
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var _StickerSerializer = sticker_serializer_1.default;
var _VideoSerializer = video_serializer_1.default;
var _VoiceSerializer = voice_serializer_1.default;
var _VideoNoteSerializer = video_note_serializer_1.default;
var _ContactSerializer = contact_serializer_1.default;
var _LocationSerializer = location_serializer_1.default;
var _VenueSerializer = venue_serializer_1.default;
var _PollSerializer = poll_serializer_1.default;
var _DiceSerializer = dice_serializer_1.default;
var _InvoiceSerializer = invoice_serializer_1.default;
var _SuccessfulPaymentSerializer = successful_payment_serializer_1.default;
var _PassportDataSerializer = passport_data_serializer_1.default;
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    messageId: {
        required: true,
        type: 'number'
    },
    from: {
        required: false,
        type: 'User'
    },
    date: {
        required: true,
        type: 'number'
    },
    chat: {
        required: true,
        type: 'Chat'
    },
    forwardFrom: {
        required: false,
        type: 'User'
    },
    forwardFromChat: {
        required: false,
        type: 'Chat'
    },
    forwardFromMessageId: {
        required: false,
        type: 'number'
    },
    forwardSignature: {
        required: false,
        type: 'string'
    },
    forwardSenderName: {
        required: false,
        type: 'string'
    },
    forwardDate: {
        required: false,
        type: 'number'
    },
    replyToMessage: {
        required: false,
        type: 'Message'
    },
    editDate: {
        required: false,
        type: 'number'
    },
    mediaGroupId: {
        required: false,
        type: 'string'
    },
    authorSignature: {
        required: false,
        type: 'string'
    },
    text: {
        required: false,
        type: 'string'
    },
    entities: {
        required: false,
        type: 'MessageEntity[]'
    },
    captionEntities: {
        required: false,
        type: 'MessageEntity[]'
    },
    audio: {
        required: false,
        type: 'Audio'
    },
    document: {
        required: false,
        type: 'Document'
    },
    animation: {
        required: false,
        type: 'Animation'
    },
    game: {
        required: false,
        type: 'Game'
    },
    photo: {
        required: false,
        type: 'PhotoSize[]'
    },
    sticker: {
        required: false,
        type: 'Sticker'
    },
    video: {
        required: false,
        type: 'Video'
    },
    voice: {
        required: false,
        type: 'Voice'
    },
    videoNote: {
        required: false,
        type: 'VideoNote'
    },
    caption: {
        required: false,
        type: 'string'
    },
    contact: {
        required: false,
        type: 'Contact'
    },
    location: {
        required: false,
        type: 'Location'
    },
    venue: {
        required: false,
        type: 'Venue'
    },
    poll: {
        required: false,
        type: 'Poll'
    },
    dice: {
        required: false,
        type: 'Dice'
    },
    newChatMembers: {
        required: false,
        type: 'User[]'
    },
    leftChatMember: {
        required: false,
        type: 'User'
    },
    newChatTitle: {
        required: false,
        type: 'string'
    },
    newChatPhoto: {
        required: false,
        type: 'PhotoSize[]'
    },
    deleteChatPhoto: {
        required: false,
        type: 'boolean'
    },
    groupChatCreated: {
        required: false,
        type: 'boolean'
    },
    supergroupChatCreated: {
        required: false,
        type: 'boolean'
    },
    channelChatCreated: {
        required: false,
        type: 'boolean'
    },
    migrateToChatId: {
        required: false,
        type: 'number'
    },
    migrateFromChatId: {
        required: false,
        type: 'number'
    },
    pinnedMessage: {
        required: false,
        type: 'Message'
    },
    invoice: {
        required: false,
        type: 'Invoice'
    },
    successfulPayment: {
        required: false,
        type: 'SuccessfulPayment'
    },
    connectedWebsite: {
        required: false,
        type: 'string'
    },
    passportData: {
        required: false,
        type: 'PassportData'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.MessageSerializerParams = params;
var MessageSerializer = new serializer_1.Serializer(message_1.default, 'Message', params);
exports.MessageSerializer = MessageSerializer;
exports.default = MessageSerializer;
