// telegram-bot-ts-types/build/serialize/poll_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var poll_1 = require("../entities/poll");
var poll_option_serializer_1 = require("./poll_option_serializer");
var _PollOptionSerializer = poll_option_serializer_1.default;
var params = {
    id: {
        required: true,
        type: 'string'
    },
    question: {
        required: true,
        type: 'string'
    },
    options: {
        required: true,
        type: 'PollOption[]'
    },
    totalVoterCount: {
        required: true,
        type: 'number'
    },
    isClosed: {
        required: true,
        type: 'boolean'
    },
    isAnonymous: {
        required: true,
        type: 'boolean'
    },
    type: {
        required: true,
        type: 'string'
    },
    allowsMultipleAnswers: {
        required: true,
        type: 'boolean'
    },
    correctOptionId: {
        required: false,
        type: 'number'
    }
};
exports.PollSerializerParams = params;
var PollSerializer = new serializer_1.Serializer(poll_1.default, 'Poll', params);
exports.PollSerializer = PollSerializer;
exports.default = PollSerializer;
