// telegram-bot-ts-types/build/serialize/pre_checkout_query_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var pre_checkout_query_1 = require("../entities/pre_checkout_query");
var user_serializer_1 = require("./user_serializer");
var order_info_serializer_1 = require("./order_info_serializer");
var _UserSerializer = user_serializer_1.default;
var _OrderInfoSerializer = order_info_serializer_1.default;
var params = {
    id: {
        required: true,
        type: 'string'
    },
    from: {
        required: true,
        type: 'User'
    },
    currency: {
        required: true,
        type: 'string'
    },
    totalAmount: {
        required: true,
        type: 'number'
    },
    invoicePayload: {
        required: true,
        type: 'string'
    },
    shippingOptionId: {
        required: false,
        type: 'string'
    },
    orderInfo: {
        required: false,
        type: 'OrderInfo'
    }
};
exports.PreCheckoutQuerySerializerParams = params;
var PreCheckoutQuerySerializer = new serializer_1.Serializer(pre_checkout_query_1.default, 'PreCheckoutQuery', params);
exports.PreCheckoutQuerySerializer = PreCheckoutQuerySerializer;
exports.default = PreCheckoutQuerySerializer;
