// telegram-bot-ts-types/build/serialize/promote_chat_member_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var promote_chat_member_param_1 = require("../method_params/promote_chat_member_param");
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    userId: {
        required: true,
        type: 'number'
    },
    canChangeInfo: {
        required: false,
        type: 'boolean'
    },
    canPostMessages: {
        required: false,
        type: 'boolean'
    },
    canEditMessages: {
        required: false,
        type: 'boolean'
    },
    canDeleteMessages: {
        required: false,
        type: 'boolean'
    },
    canInviteUsers: {
        required: false,
        type: 'boolean'
    },
    canRestrictMembers: {
        required: false,
        type: 'boolean'
    },
    canPinMessages: {
        required: false,
        type: 'boolean'
    },
    canPromoteMembers: {
        required: false,
        type: 'boolean'
    }
};
exports.PromoteChatMemberParamSerializerParams = params;
var PromoteChatMemberParamSerializer = new serializer_1.Serializer(promote_chat_member_param_1.default, 'PromoteChatMemberParam', params);
exports.PromoteChatMemberParamSerializer = PromoteChatMemberParamSerializer;
exports.default = PromoteChatMemberParamSerializer;
