// telegram-bot-ts-types/build/serialize/restrict_chat_member_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var restrict_chat_member_param_1 = require("../method_params/restrict_chat_member_param");
var chat_permissions_serializer_1 = require("./chat_permissions_serializer");
var _ChatPermissionsSerializer = chat_permissions_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    userId: {
        required: true,
        type: 'number'
    },
    permissions: {
        required: true,
        type: 'ChatPermissions'
    },
    untilDate: {
        required: false,
        type: 'number'
    }
};
exports.RestrictChatMemberParamSerializerParams = params;
var RestrictChatMemberParamSerializer = new serializer_1.Serializer(restrict_chat_member_param_1.default, 'RestrictChatMemberParam', params);
exports.RestrictChatMemberParamSerializer = RestrictChatMemberParamSerializer;
exports.default = RestrictChatMemberParamSerializer;
