// telegram-bot-ts-types/build/serialize/send_video_note_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_video_note_param_1 = require("../method_params/send_video_note_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var reply_keyboard_markup_serializer_1 = require("./reply_keyboard_markup_serializer");
var reply_keyboard_remove_serializer_1 = require("./reply_keyboard_remove_serializer");
var force_reply_serializer_1 = require("./force_reply_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _ReplyKeyboardMarkupSerializer = reply_keyboard_markup_serializer_1.default;
var _ReplyKeyboardRemoveSerializer = reply_keyboard_remove_serializer_1.default;
var _ForceReplySerializer = force_reply_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    videoNote: {
        required: true,
        type: 'InputFile | string'
    },
    duration: {
        required: false,
        type: 'number'
    },
    length: {
        required: false,
        type: 'number'
    },
    thumb: {
        required: false,
        type: 'InputFile | string'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply'
    }
};
exports.SendVideoNoteParamSerializerParams = params;
var SendVideoNoteParamSerializer = new serializer_1.Serializer(send_video_note_param_1.default, 'SendVideoNoteParam', params);
exports.SendVideoNoteParamSerializer = SendVideoNoteParamSerializer;
exports.default = SendVideoNoteParamSerializer;
