// telegram-bot-ts-types/build/serialize/send_invoice_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_invoice_param_1 = require("../method_params/send_invoice_param");
var labeled_price_serializer_1 = require("./labeled_price_serializer");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _LabeledPriceSerializer = labeled_price_serializer_1.default;
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number'
    },
    title: {
        required: true,
        type: 'string'
    },
    description: {
        required: true,
        type: 'string'
    },
    payload: {
        required: true,
        type: 'string'
    },
    providerToken: {
        required: true,
        type: 'string'
    },
    startParameter: {
        required: true,
        type: 'string'
    },
    currency: {
        required: true,
        type: 'string'
    },
    prices: {
        required: true,
        type: 'LabeledPrice[]'
    },
    providerData: {
        required: false,
        type: 'string'
    },
    photoUrl: {
        required: false,
        type: 'string'
    },
    photoSize: {
        required: false,
        type: 'number'
    },
    photoWidth: {
        required: false,
        type: 'number'
    },
    photoHeight: {
        required: false,
        type: 'number'
    },
    needName: {
        required: false,
        type: 'boolean'
    },
    needPhoneNumber: {
        required: false,
        type: 'boolean'
    },
    needEmail: {
        required: false,
        type: 'boolean'
    },
    needShippingAddress: {
        required: false,
        type: 'boolean'
    },
    sendPhoneNumberToProvider: {
        required: false,
        type: 'boolean'
    },
    sendEmailToProvider: {
        required: false,
        type: 'boolean'
    },
    isFlexible: {
        required: false,
        type: 'boolean'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.SendInvoiceParamSerializerParams = params;
var SendInvoiceParamSerializer = new serializer_1.Serializer(send_invoice_param_1.default, 'SendInvoiceParam', params);
exports.SendInvoiceParamSerializer = SendInvoiceParamSerializer;
exports.default = SendInvoiceParamSerializer;
