// telegram-bot-ts-types/build/serialize/send_venue_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_venue_param_1 = require("../method_params/send_venue_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var reply_keyboard_markup_serializer_1 = require("./reply_keyboard_markup_serializer");
var reply_keyboard_remove_serializer_1 = require("./reply_keyboard_remove_serializer");
var force_reply_serializer_1 = require("./force_reply_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _ReplyKeyboardMarkupSerializer = reply_keyboard_markup_serializer_1.default;
var _ReplyKeyboardRemoveSerializer = reply_keyboard_remove_serializer_1.default;
var _ForceReplySerializer = force_reply_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    latitude: {
        required: true,
        type: 'number'
    },
    longitude: {
        required: true,
        type: 'number'
    },
    title: {
        required: true,
        type: 'string'
    },
    address: {
        required: true,
        type: 'string'
    },
    foursquareId: {
        required: false,
        type: 'string'
    },
    foursquareType: {
        required: false,
        type: 'string'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply'
    }
};
exports.SendVenueParamSerializerParams = params;
var SendVenueParamSerializer = new serializer_1.Serializer(send_venue_param_1.default, 'SendVenueParam', params);
exports.SendVenueParamSerializer = SendVenueParamSerializer;
exports.default = SendVenueParamSerializer;
