// telegram-bot-ts-types/build/serialize/send_document_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_document_param_1 = require("../method_params/send_document_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var reply_keyboard_markup_serializer_1 = require("./reply_keyboard_markup_serializer");
var reply_keyboard_remove_serializer_1 = require("./reply_keyboard_remove_serializer");
var force_reply_serializer_1 = require("./force_reply_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _ReplyKeyboardMarkupSerializer = reply_keyboard_markup_serializer_1.default;
var _ReplyKeyboardRemoveSerializer = reply_keyboard_remove_serializer_1.default;
var _ForceReplySerializer = force_reply_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    document: {
        required: true,
        type: 'InputFile | string'
    },
    thumb: {
        required: false,
        type: 'InputFile | string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply'
    }
};
exports.SendDocumentParamSerializerParams = params;
var SendDocumentParamSerializer = new serializer_1.Serializer(send_document_param_1.default, 'SendDocumentParam', params);
exports.SendDocumentParamSerializer = SendDocumentParamSerializer;
exports.default = SendDocumentParamSerializer;
