// telegram-bot-ts-types/build/serialize/send_game_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_game_param_1 = require("../method_params/send_game_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number'
    },
    gameShortName: {
        required: true,
        type: 'string'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.SendGameParamSerializerParams = params;
var SendGameParamSerializer = new serializer_1.Serializer(send_game_param_1.default, 'SendGameParam', params);
exports.SendGameParamSerializer = SendGameParamSerializer;
exports.default = SendGameParamSerializer;
