// telegram-bot-ts-types/build/serialize/send_poll_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_poll_param_1 = require("../method_params/send_poll_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var reply_keyboard_markup_serializer_1 = require("./reply_keyboard_markup_serializer");
var reply_keyboard_remove_serializer_1 = require("./reply_keyboard_remove_serializer");
var force_reply_serializer_1 = require("./force_reply_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _ReplyKeyboardMarkupSerializer = reply_keyboard_markup_serializer_1.default;
var _ReplyKeyboardRemoveSerializer = reply_keyboard_remove_serializer_1.default;
var _ForceReplySerializer = force_reply_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    question: {
        required: true,
        type: 'string'
    },
    options: {
        required: true,
        type: 'string[]'
    },
    isAnonymous: {
        required: false,
        type: 'boolean'
    },
    type: {
        required: false,
        type: 'string'
    },
    allowsMultipleAnswers: {
        required: false,
        type: 'boolean'
    },
    correctOptionId: {
        required: false,
        type: 'number'
    },
    isClosed: {
        required: false,
        type: 'boolean'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply'
    }
};
exports.SendPollParamSerializerParams = params;
var SendPollParamSerializer = new serializer_1.Serializer(send_poll_param_1.default, 'SendPollParam', params);
exports.SendPollParamSerializer = SendPollParamSerializer;
exports.default = SendPollParamSerializer;
