// telegram-bot-ts-types/build/serialize/send_video_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_video_param_1 = require("../method_params/send_video_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var reply_keyboard_markup_serializer_1 = require("./reply_keyboard_markup_serializer");
var reply_keyboard_remove_serializer_1 = require("./reply_keyboard_remove_serializer");
var force_reply_serializer_1 = require("./force_reply_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var _ReplyKeyboardMarkupSerializer = reply_keyboard_markup_serializer_1.default;
var _ReplyKeyboardRemoveSerializer = reply_keyboard_remove_serializer_1.default;
var _ForceReplySerializer = force_reply_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    video: {
        required: true,
        type: 'InputFile | string'
    },
    duration: {
        required: false,
        type: 'number'
    },
    width: {
        required: false,
        type: 'number'
    },
    height: {
        required: false,
        type: 'number'
    },
    thumb: {
        required: false,
        type: 'InputFile | string'
    },
    caption: {
        required: false,
        type: 'string'
    },
    parseMode: {
        required: false,
        type: 'string'
    },
    supportsStreaming: {
        required: false,
        type: 'boolean'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply'
    }
};
exports.SendVideoParamSerializerParams = params;
var SendVideoParamSerializer = new serializer_1.Serializer(send_video_param_1.default, 'SendVideoParam', params);
exports.SendVideoParamSerializer = SendVideoParamSerializer;
exports.default = SendVideoParamSerializer;
