// telegram-bot-ts-types/build/serialize/send_media_group_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var send_media_group_param_1 = require("../method_params/send_media_group_param");
var input_media_photo_serializer_1 = require("./input_media_photo_serializer");
var input_media_video_serializer_1 = require("./input_media_video_serializer");
var _InputMediaPhotoSerializer = input_media_photo_serializer_1.default;
var _InputMediaVideoSerializer = input_media_video_serializer_1.default;
var params = {
    chatId: {
        required: true,
        type: 'number | string'
    },
    media: {
        required: true,
        type: 'InputMediaPhoto[] | InputMediaVideo[]'
    },
    disableNotification: {
        required: false,
        type: 'boolean'
    },
    replyToMessageId: {
        required: false,
        type: 'number'
    }
};
exports.SendMediaGroupParamSerializerParams = params;
var SendMediaGroupParamSerializer = new serializer_1.Serializer(send_media_group_param_1.default, 'SendMediaGroupParam', params);
exports.SendMediaGroupParamSerializer = SendMediaGroupParamSerializer;
exports.default = SendMediaGroupParamSerializer;
