// telegram-bot-ts-types/build/serialize/set_game_score_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var set_game_score_param_1 = require("../method_params/set_game_score_param");
var params = {
    userId: {
        required: true,
        type: 'number'
    },
    score: {
        required: true,
        type: 'number'
    },
    force: {
        required: false,
        type: 'boolean'
    },
    disableEditMessage: {
        required: false,
        type: 'boolean'
    },
    chatId: {
        required: false,
        type: 'number'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    }
};
exports.SetGameScoreParamSerializerParams = params;
var SetGameScoreParamSerializer = new serializer_1.Serializer(set_game_score_param_1.default, 'SetGameScoreParam', params);
exports.SetGameScoreParamSerializer = SetGameScoreParamSerializer;
exports.default = SetGameScoreParamSerializer;
