// telegram-bot-ts-types/build/serialize/shipping_query_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var shipping_query_1 = require("../entities/shipping_query");
var user_serializer_1 = require("./user_serializer");
var shipping_address_serializer_1 = require("./shipping_address_serializer");
var _UserSerializer = user_serializer_1.default;
var _ShippingAddressSerializer = shipping_address_serializer_1.default;
var params = {
    id: {
        required: true,
        type: 'string'
    },
    from: {
        required: true,
        type: 'User'
    },
    invoicePayload: {
        required: true,
        type: 'string'
    },
    shippingAddress: {
        required: true,
        type: 'ShippingAddress'
    }
};
exports.ShippingQuerySerializerParams = params;
var ShippingQuerySerializer = new serializer_1.Serializer(shipping_query_1.default, 'ShippingQuery', params);
exports.ShippingQuerySerializer = ShippingQuerySerializer;
exports.default = ShippingQuerySerializer;
