// telegram-bot-ts-types/build/serialize/sticker_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var sticker_1 = require("../entities/sticker");
var photo_size_serializer_1 = require("./photo_size_serializer");
var mask_position_serializer_1 = require("./mask_position_serializer");
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var _MaskPositionSerializer = mask_position_serializer_1.default;
var params = {
    fileId: {
        required: true,
        type: 'string'
    },
    fileUniqueId: {
        required: true,
        type: 'string'
    },
    width: {
        required: true,
        type: 'number'
    },
    height: {
        required: true,
        type: 'number'
    },
    isAnimated: {
        required: true,
        type: 'boolean'
    },
    thumb: {
        required: false,
        type: 'PhotoSize'
    },
    emoji: {
        required: false,
        type: 'string'
    },
    setName: {
        required: false,
        type: 'string'
    },
    maskPosition: {
        required: false,
        type: 'MaskPosition'
    },
    fileSize: {
        required: false,
        type: 'number'
    }
};
exports.StickerSerializerParams = params;
var StickerSerializer = new serializer_1.Serializer(sticker_1.default, 'Sticker', params);
exports.StickerSerializer = StickerSerializer;
exports.default = StickerSerializer;
