// telegram-bot-ts-types/build/serialize/sticker_set_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var sticker_set_1 = require("../entities/sticker_set");
var sticker_serializer_1 = require("./sticker_serializer");
var photo_size_serializer_1 = require("./photo_size_serializer");
var _StickerSerializer = sticker_serializer_1.default;
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var params = {
    name: {
        required: true,
        type: 'string'
    },
    title: {
        required: true,
        type: 'string'
    },
    isAnimated: {
        required: true,
        type: 'boolean'
    },
    containsMasks: {
        required: true,
        type: 'boolean'
    },
    stickers: {
        required: true,
        type: 'Sticker[]'
    },
    thumb: {
        required: false,
        type: 'PhotoSize'
    }
};
exports.StickerSetSerializerParams = params;
var StickerSetSerializer = new serializer_1.Serializer(sticker_set_1.default, 'StickerSet', params);
exports.StickerSetSerializer = StickerSetSerializer;
exports.default = StickerSetSerializer;
