// telegram-bot-ts-types/build/serialize/stop_message_live_location_param_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var stop_message_live_location_param_1 = require("../method_params/stop_message_live_location_param");
var inline_keyboard_markup_serializer_1 = require("./inline_keyboard_markup_serializer");
var _InlineKeyboardMarkupSerializer = inline_keyboard_markup_serializer_1.default;
var params = {
    chatId: {
        required: false,
        type: 'number | string'
    },
    messageId: {
        required: false,
        type: 'number'
    },
    inlineMessageId: {
        required: false,
        type: 'string'
    },
    replyMarkup: {
        required: false,
        type: 'InlineKeyboardMarkup'
    }
};
exports.StopMessageLiveLocationParamSerializerParams = params;
var StopMessageLiveLocationParamSerializer = new serializer_1.Serializer(stop_message_live_location_param_1.default, 'StopMessageLiveLocationParam', params);
exports.StopMessageLiveLocationParamSerializer = StopMessageLiveLocationParamSerializer;
exports.default = StopMessageLiveLocationParamSerializer;
