// telegram-bot-ts-types/build/serialize/successful_payment_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var successful_payment_1 = require("../entities/successful_payment");
var order_info_serializer_1 = require("./order_info_serializer");
var _OrderInfoSerializer = order_info_serializer_1.default;
var params = {
    currency: {
        required: true,
        type: 'string'
    },
    totalAmount: {
        required: true,
        type: 'number'
    },
    invoicePayload: {
        required: true,
        type: 'string'
    },
    shippingOptionId: {
        required: false,
        type: 'string'
    },
    orderInfo: {
        required: false,
        type: 'OrderInfo'
    },
    telegramPaymentChargeId: {
        required: true,
        type: 'string'
    },
    providerPaymentChargeId: {
        required: true,
        type: 'string'
    }
};
exports.SuccessfulPaymentSerializerParams = params;
var SuccessfulPaymentSerializer = new serializer_1.Serializer(successful_payment_1.default, 'SuccessfulPayment', params);
exports.SuccessfulPaymentSerializer = SuccessfulPaymentSerializer;
exports.default = SuccessfulPaymentSerializer;
