// telegram-bot-ts-types/build/serialize/update_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var update_1 = require("../entities/update");
var message_serializer_1 = require("./message_serializer");
var inline_query_serializer_1 = require("./inline_query_serializer");
var chosen_inline_result_serializer_1 = require("./chosen_inline_result_serializer");
var callback_query_serializer_1 = require("./callback_query_serializer");
var shipping_query_serializer_1 = require("./shipping_query_serializer");
var pre_checkout_query_serializer_1 = require("./pre_checkout_query_serializer");
var poll_serializer_1 = require("./poll_serializer");
var poll_answer_serializer_1 = require("./poll_answer_serializer");
var _MessageSerializer = message_serializer_1.default;
var _InlineQuerySerializer = inline_query_serializer_1.default;
var _ChosenInlineResultSerializer = chosen_inline_result_serializer_1.default;
var _CallbackQuerySerializer = callback_query_serializer_1.default;
var _ShippingQuerySerializer = shipping_query_serializer_1.default;
var _PreCheckoutQuerySerializer = pre_checkout_query_serializer_1.default;
var _PollSerializer = poll_serializer_1.default;
var _PollAnswerSerializer = poll_answer_serializer_1.default;
var params = {
    updateId: {
        required: true,
        type: 'number'
    },
    message: {
        required: false,
        type: 'Message'
    },
    editedMessage: {
        required: false,
        type: 'Message'
    },
    channelPost: {
        required: false,
        type: 'Message'
    },
    editedChannelPost: {
        required: false,
        type: 'Message'
    },
    inlineQuery: {
        required: false,
        type: 'InlineQuery'
    },
    chosenInlineResult: {
        required: false,
        type: 'ChosenInlineResult'
    },
    callbackQuery: {
        required: false,
        type: 'CallbackQuery'
    },
    shippingQuery: {
        required: false,
        type: 'ShippingQuery'
    },
    preCheckoutQuery: {
        required: false,
        type: 'PreCheckoutQuery'
    },
    poll: {
        required: false,
        type: 'Poll'
    },
    pollAnswer: {
        required: false,
        type: 'PollAnswer'
    }
};
exports.UpdateSerializerParams = params;
var UpdateSerializer = new serializer_1.Serializer(update_1.default, 'Update', params);
exports.UpdateSerializer = UpdateSerializer;
exports.default = UpdateSerializer;
