// telegram-bot-ts-types/build/serialize/user_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var user_1 = require("../entities/user");
var params = {
    id: {
        required: true,
        type: 'number'
    },
    isBot: {
        required: true,
        type: 'boolean'
    },
    firstName: {
        required: true,
        type: 'string'
    },
    lastName: {
        required: false,
        type: 'string'
    },
    username: {
        required: false,
        type: 'string'
    },
    languageCode: {
        required: false,
        type: 'string'
    },
    canJoinGroups: {
        required: false,
        type: 'boolean'
    },
    canReadAllGroupMessages: {
        required: false,
        type: 'boolean'
    },
    supportsInlineQueries: {
        required: false,
        type: 'boolean'
    }
};
exports.UserSerializerParams = params;
var UserSerializer = new serializer_1.Serializer(user_1.default, 'User', params);
exports.UserSerializer = UserSerializer;
exports.default = UserSerializer;
