// telegram-bot-ts-types/build/serialize/video_serializer.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var serializer_1 = require("./serializer");
var video_1 = require("../entities/video");
var photo_size_serializer_1 = require("./photo_size_serializer");
var _PhotoSizeSerializer = photo_size_serializer_1.default;
var params = {
    fileId: {
        required: true,
        type: 'string'
    },
    fileUniqueId: {
        required: true,
        type: 'string'
    },
    width: {
        required: true,
        type: 'number'
    },
    height: {
        required: true,
        type: 'number'
    },
    duration: {
        required: true,
        type: 'number'
    },
    thumb: {
        required: false,
        type: 'PhotoSize'
    },
    mimeType: {
        required: false,
        type: 'string'
    },
    fileSize: {
        required: false,
        type: 'number'
    }
};
exports.VideoSerializerParams = params;
var VideoSerializer = new serializer_1.Serializer(video_1.default, 'Video', params);
exports.VideoSerializer = VideoSerializer;
exports.default = VideoSerializer;
