// telegram-bot-ts/build/src/core/failures.js
"use strict";
var __extends = (this && this.__extends) || (function () {
    var extendStatics = function (d, b) {
        extendStatics = Object.setPrototypeOf ||
            ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
            function (d, b) { for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p]; };
        return extendStatics(d, b);
    };
    return function (d, b) {
        extendStatics(d, b);
        function __() { this.constructor = d; }
        d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
    };
})();
Object.defineProperty(exports, "__esModule", { value: true });
var Failure = /** @class */ (function () {
    function Failure(message) {
        this.message = message !== null && message !== void 0 ? message : 'Error';
    }
    Failure.prototype.getMessage = function () {
        return this.message;
    };
    return Failure;
}());
exports.Failure = Failure;
var FailureApi = /** @class */ (function (_super) {
    __extends(FailureApi, _super);
    function FailureApi() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
    return FailureApi;
}(Failure));
exports.FailureApi = FailureApi;
var FailureUnauthorized = /** @class */ (function (_super) {
    __extends(FailureUnauthorized, _super);
    function FailureUnauthorized() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
    return FailureUnauthorized;
}(Failure));
exports.FailureUnauthorized = FailureUnauthorized;
var FailureWrongStructure = /** @class */ (function (_super) {
    __extends(FailureWrongStructure, _super);
    function FailureWrongStructure() {
        return _super !== null && _super.apply(this, arguments) || this;
    }
    return FailureWrongStructure;
}(Failure));
exports.FailureWrongStructure = FailureWrongStructure;
